diff --git a/content/browser/web_contents/web_contents_view_mac.h b/content/browser/web_contents/web_contents_view_mac.h
index b9868665c8..8de3e6da9c 100644
--- a/content/browser/web_contents/web_contents_view_mac.h
+++ b/content/browser/web_contents/web_contents_view_mac.h
@@ -183,14 +183,16 @@
                           base::FilePath* out_file_path) override;
   void EndDrag(uint32_t drag_operation,
                const gfx::PointF& local_point,
-               const gfx::PointF& screen_point) override;
+               const gfx::PointF& screen_point,
+               bool ended_by_mouse_release) override;
 
   // Helper used by `EndDrag` to pass a callback to `drag_dest_`. This can be
   // called either synchronously or asynchronously depending on if `drag_dest_`
   // delays firing "dragend" or not.
   void PerformEndDrag(uint32_t drag_operation,
                       const gfx::PointF& local_point,
-                      const gfx::PointF& screen_point);
+                      const gfx::PointF& screen_point,
+                      bool ended_by_mouse_release);
 
   // remote_cocoa::mojom::WebContentsNSViewHost, synchronous methods:
   void DraggingEntered(remote_cocoa::mojom::DraggingInfoPtr dragging_info,
