diff --git a/chrome/browser/picture_in_picture/picture_in_picture_window_manager_unittest.cc b/chrome/browser/picture_in_picture/picture_in_picture_window_manager_unittest.cc
index 494278bd4e..927936d54e 100644
--- a/chrome/browser/picture_in_picture/picture_in_picture_window_manager_unittest.cc
+++ b/chrome/browser/picture_in_picture/picture_in_picture_window_manager_unittest.cc
@@ -187,10 +187,15 @@ TEST_F(PictureInPictureWindowManagerTest, OnEnterVideoPictureInPicture) {
 
 #if !BUILDFLAG(IS_ANDROID)
 TEST_F(PictureInPictureWindowManagerTest, RespectsMinAndMaxSize) {
-  // The max window size should be 80% of the screen.
+  // The max window size should use the complete display work area.
   display::Display display(/*id=*/1, gfx::Rect(0, 0, 1000, 1000));
-  EXPECT_EQ(gfx::Size(800, 800),
+  display.set_work_area(gfx::Rect(0, 24, 1000, 976));
+  EXPECT_EQ(gfx::Size(1000, 976),
             PictureInPictureWindowManager::GetMaximumWindowSize(display));
 
+  EXPECT_EQ(gfx::Size(800, 100),
+            PictureInPictureWindowManager::AdjustRequestedSizeIfNecessary(
+                gfx::Size(900, 100), display));
+
   // The initial bounds of the PiP window should respect that.
   blink::mojom::PictureInPictureWindowOptions pip_options;
