diff --git a/chrome/browser/extensions/api/settings_private/prefs_util.cc b/chrome/browser/extensions/api/settings_private/prefs_util.cc
index 7ef1c895e0..0000000001 100644
--- a/chrome/browser/extensions/api/settings_private/prefs_util.cc
+++ b/chrome/browser/extensions/api/settings_private/prefs_util.cc
@@ -16,6 +16,7 @@
 #include "chrome/browser/content_settings/generated_javascript_optimizer_pref.h"
 #include "chrome/browser/content_settings/generated_permission_prompting_behavior_pref.h"
 #include "chrome/browser/extensions/api/settings_private/generated_prefs.h"
+#include "dao/browser/dao_pref_names.h"
 #include "chrome/browser/extensions/api/settings_private/generated_prefs_factory.h"
 #include "chrome/browser/extensions/settings_api_helpers.h"
 #include "chrome/browser/glic/glic_pref_names.h"
@@ -236,5 +237,11 @@ const PrefsUtil::TypedPrefMap& PrefsUtil::GetAllowlistedKeys() {
   (*s_allowlist)[payments::kCanMakePaymentEnabled] =
       settings_api::PrefType::kBoolean;
+  (*s_allowlist)[dao::prefs::kDaoLittleDaoEnabled] =
+      settings_api::PrefType::kBoolean;
+  (*s_allowlist)[dao::prefs::kDaoEnhancedPipEnabled] =
+      settings_api::PrefType::kBoolean;
+  (*s_allowlist)[dao::prefs::kDaoEnhancedCommandBarSuggestionsEnabled] =
+      settings_api::PrefType::kBoolean;
   (*s_allowlist)[bookmarks::prefs::kShowBookmarkBar] =
       settings_api::PrefType::kBoolean;
   (*s_allowlist)[bookmarks::prefs::kShowTabGroupsInBookmarkBar] =
