diff --git a/components/global_media_controls/public/views/media_progress_view.cc b/components/global_media_controls/public/views/media_progress_view.cc
index 3f3e1d65b3..8a5d4a754a 100644
--- a/components/global_media_controls/public/views/media_progress_view.cc
+++ b/components/global_media_controls/public/views/media_progress_view.cc
@@ -55,7 +55,7 @@ constexpr int kProgressPhaseSpeed = 28;
 constexpr gfx::SizeF kSquigglyProgressIndicatorSize = gfx::SizeF(6, 14);
 constexpr gfx::SizeF kStraightProgressIndicatorSize = gfx::SizeF(4, 16);
 
-// The width of the gap between the progress indicator and the straight lines.
+// Gap in pixels between the straight progress indicator and background line.
 constexpr int kStraightProgressIndicatorGap = 4;
 
 // Defines how long the animation for progress transitioning between squiggly
@@ -246,9 +246,7 @@ void MediaProgressView::OnPaint(gfx::Canvas* canvas) {
   } else {
     // Paint the foreground straight progress line with rounded corners.
     flags.setStyle(cc::PaintFlags::kFill_Style);
-    const int foreground_progress_width =
-        progress_width - kStraightProgressIndicatorGap -
-        kStraightProgressIndicatorSize.width() / 2;
+    const int foreground_progress_width = progress_width;
     if (foreground_progress_width > 0) {
       canvas->DrawRoundRect(
           gfx::RectF(0, (view_height - straight_progress_stroke_width_) / 2,
