diff --git a/chrome/browser/ui/webui/settings/settings_ui.cc b/chrome/browser/ui/webui/settings/settings_ui.cc
index ecb360f35259e..fdc235dbb4577 100644
--- a/chrome/browser/ui/webui/settings/settings_ui.cc
+++ b/chrome/browser/ui/webui/settings/settings_ui.cc
@@ -300,9 +300,7 @@ SettingsUI::SettingsUI(content::WebUI* web_ui)
   AddSettingsPageUIHandler(std::make_unique<MacSystemSettingsHandler>());
 #endif
 
-  html_source->AddBoolean("signinAllowed", !profile->IsGuestSession() &&
-                                               profile->GetPrefs()->GetBoolean(
-                                                   prefs::kSigninAllowed));
+  html_source->AddBoolean("signinAllowed", false);
 
   html_source->AddBoolean(
       "shouldUseMetricsConsentRestructure",
