diff --git a/components/error_page_strings.grdp b/components/error_page_strings.grdp
index b32621b13e..88ab84ce04 100644
--- a/components/error_page_strings.grdp
+++ b/components/error_page_strings.grdp
@@ -89,7 +89,7 @@
     Check your administrator's policies
   </message>
   <message name="IDS_ERRORPAGES_SUGGESTION_VIEW_POLICIES_BODY" desc="When a webpage fails to load, sometimes we suggest to the user to view the current policies.  These are instructions below a header.">
-    Visit &lt;strong&gt;chrome://policy&lt;/strong&gt; to see the list of blocked URLs and other policies enforced by your system administrator.
+    Visit &lt;strong&gt;dao://policy&lt;/strong&gt; to see the list of blocked URLs and other policies enforced by your system administrator.
   </message>
   <message name="IDS_ERRORPAGES_SUGGESTION_UNSUPPORTED_CIPHER_HEADER" desc="This is the header displayed above the explanation for SSL cipher and version errors.">
     Unsupported protocol
