diff --git a/components/flags_strings.grdp b/components/flags_strings.grdp
index 85ff523564..620f623532 100644
--- a/components/flags_strings.grdp
+++ b/components/flags_strings.grdp
@@ -55,7 +55,7 @@
       Relaunch
     </message>
   </if>
-  <message name="IDS_FLAGS_UI_CLEAR_SEARCH" desc="Tooltip for icon button that clears the searchbar on the chrome://flags page.">
+  <message name="IDS_FLAGS_UI_CLEAR_SEARCH" desc="Tooltip for icon button that clears the searchbar on the dao://flags page.">
     Clear search
   </message>
   <message name="IDS_FLAGS_UI_RESET_ACKNOWLEDGED" desc="Screen reader text stating that the reset action was acknowledged.">
