diff --git a/components/policy_strings.grdp b/components/policy_strings.grdp
index 293e50152b..e155364327 100644
--- a/components/policy_strings.grdp
+++ b/components/policy_strings.grdp
@@ -467,7 +467,7 @@ Additional details:
     The URL of the shortcut exceeds the maximum allowed length of <ph name="MAX_LENGTH">$1<ex>1000</ex></ph> characters.
   </message>
 
-  <!-- chrome://policy -->
+  <!-- dao://policy -->
   <message name="IDS_POLICY_TITLE" desc="Page title and the title of the section that lists policies.">
     Policies
   </message>
@@ -572,10 +572,10 @@ Additional details:
   <message name="IDS_POLICY_LABEL_PUSH_POLICIES" desc="Label to indicates the policies can be pushed or not to chrome.">
     Policies push:
   </message>
-  <message name="IDS_POLICY_PUSH_POLICIES_ON" desc="Label in the chrome://policy information page." meaning="Policies can be pushed to chrome.">
+  <message name="IDS_POLICY_PUSH_POLICIES_ON" desc="Label in the dao://policy information page." meaning="Policies can be pushed to chrome.">
     On
   </message>
-  <message name="IDS_POLICY_PUSH_POLICIES_OFF" desc="Label in the chrome://policy information page." meaning="Policies cannot be pushed to chrome.">
+  <message name="IDS_POLICY_PUSH_POLICIES_OFF" desc="Label in the dao://policy information page." meaning="Policies cannot be pushed to chrome.">
     Off
   </message>
   <message name="IDS_POLICY_NEVER_FETCHED" desc="Indicates that a policy fetch was never performed before.">
@@ -839,10 +839,10 @@ Additional details:
   <message name="IDS_COPY_POLICIES_DONE" desc="Toast notification message indicating policies are copied to clipboard.">
     Copied to clipboard
   </message>
-  <message name="IDS_VIEW_POLICY_LOGS" desc="Label for the button that redirects from chrome://policy to chrome://policy/logs.">
+  <message name="IDS_VIEW_POLICY_LOGS" desc="Label for the button that redirects from dao://policy to dao://policy/logs.">
     View logs
   </message>
-  <message name="IDS_POLICY_MORE_ACTIONS" desc="Label for the dropdown menu on the chrome://policy page that offers additional actions such as downloading policy data as a JSON file, uploading report or showing policy logs.">
+  <message name="IDS_POLICY_MORE_ACTIONS" desc="Label for the dropdown menu on the dao://policy page that offers additional actions such as downloading policy data as a JSON file, uploading report or showing policy logs.">
     More actions
   </message>
   <message name="IDS_POLICY_TABLE_COLUMN_SORT" desc="Part of the label for policy table sorting arrows.">
@@ -870,7 +870,7 @@ Additional details:
     DevTools not allowed on this page
   </message>
 
-  <!-- Strings for chrome://policy/logs -->
+  <!-- Strings for dao://policy/logs -->
   <message name="IDS_POLICY_LOGS_TITLE" desc="Title for the page.">
     Policy Logs
   </message>
@@ -884,7 +884,7 @@ Additional details:
     Refresh Logs
   </message>
 
-<!-- Strings for chrome://policy/test -->
+<!-- Strings for dao://policy/test -->
   <message name="IDS_POLICY_TEST_TITLE" desc="Title for the page.">
     Policy Tests
   </message>
@@ -916,7 +916,7 @@ Additional details:
     Preset
   </message>
   <message name="IDS_LOCAL_TEST_POLICIES_ENABLED" desc="Text for banner warning user of test policies being applied">
-    Test policies have been applied. Review and reset them at chrome://policy/test. Admin policies will not be applied.
+    Test policies have been applied. Review and reset them at dao://policy/test. Admin policies will not be applied.
   </message>
   <message name="IDS_POLICY_TEST_PRESET_CUSTOM" desc="Text for the custom option of the preset dropdown in the policy test table.">
     Custom
@@ -931,7 +931,7 @@ Additional details:
   User affiliated
   </message>
 
-  <!-- Strings for Disabled chrome://.* screen -->
+  <!-- Strings for Disabled dao://.* screen -->
   <message name="IDS_CHROME_URLS_DISABLED_PAGE_HEADER" desc="A heading for the user when the application is blocked.">
     Application is blocked
   </message>
