diff --git a/components/privacy_sandbox_chrome_strings.grdp b/components/privacy_sandbox_chrome_strings.grdp
index e7862d3a78..a410b48c8b 100644
--- a/components/privacy_sandbox_chrome_strings.grdp
+++ b/components/privacy_sandbox_chrome_strings.grdp
@@ -41,7 +41,7 @@
 
   <!-- Privacy Sandbox v4 - Topics settings page. -->
   <!-- Privacy Sandbox Settings 4 - Topics Page -->
-  <message name="IDS_SETTINGS_TOPICS_PAGE_TITLE" desc="A page title and the name of a new setting. **** CONTEXT PRIVACY SANDBOX **** Chrome’s Privacy Sandbox initiative 1) deprecates third-party cookies in Chrome, 2) supports free and open content on the web (by finding better ways to support ads online), 3) while providing stronger privacy protections for users. You can see a high-level description of this public project at www.privacysanbox.com. **** NEW SETTINGS SECTION IN CHROME **** 3 new ad-specific settings appear on an “Ad privacy” page in Chrome settings. For an equivalent structure, see “Security” on chrome://settings/privacy that opens chrome://settings/security. Likewise, “Ad privacy” on chrome://settings/privacy will open chrome://settings/AdPrivacy." formatter_data="android_java">
+  <message name="IDS_SETTINGS_TOPICS_PAGE_TITLE" desc="A page title and the name of a new setting. **** CONTEXT PRIVACY SANDBOX **** Chrome’s Privacy Sandbox initiative 1) deprecates third-party cookies in Chrome, 2) supports free and open content on the web (by finding better ways to support ads online), 3) while providing stronger privacy protections for users. You can see a high-level description of this public project at www.privacysanbox.com. **** NEW SETTINGS SECTION IN CHROME **** 3 new ad-specific settings appear on an “Ad privacy” page in Chrome settings. For an equivalent structure, see “Security” on dao://settings/privacy that opens dao://settings/security. Likewise, “Ad privacy” on dao://settings/privacy will open dao://settings/AdPrivacy." formatter_data="android_java">
     Ad topics
   </message>
   <message name="IDS_SETTINGS_TOPICS_PAGE_TOGGLE_LABEL" desc="The label for a new setting." formatter_data="android_java">
