diff --git a/components/site_settings_strings.grdp b/components/site_settings_strings.grdp
index 34aafce3f8..d83f35245a 100644
--- a/components/site_settings_strings.grdp
+++ b/components/site_settings_strings.grdp
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <grit-part>
   <!-- Site Settings common -->
-  <message name="IDS_SITE_SETTINGS_TYPE_ADS" desc="This string is the name of the intrusive ads setting (visit chrome://settings/content and click 'Additional content settings').">
+  <message name="IDS_SITE_SETTINGS_TYPE_ADS" desc="This string is the name of the intrusive ads setting (visit dao://settings/content and click 'Additional content settings').">
     Intrusive ads
   </message>
-  <message name="IDS_SITE_SETTINGS_TYPE_ADS_MID_SENTENCE" desc="This string is the name of the intrusive ads setting (visit chrome://settings/content and click 'Additional content settings') when used mid-sentence.">
+  <message name="IDS_SITE_SETTINGS_TYPE_ADS_MID_SENTENCE" desc="This string is the name of the intrusive ads setting (visit dao://settings/content and click 'Additional content settings') when used mid-sentence.">
     intrusive ads
   </message>
   <message name="IDS_SITE_SETTINGS_TYPE_ANTI_ABUSE" desc="The label used for the anti-abuse site settings controls.">
