diff --git a/components/version_ui_strings.grdp b/components/version_ui_strings.grdp
index 7c3d198ccd..6d28a80972 100644
--- a/components/version_ui_strings.grdp
+++ b/components/version_ui_strings.grdp
@@ -9,30 +9,30 @@
   <message name="IDS_VERSION_UI_UNOFFICIAL" desc="unofficial build on the about:version page">
     Developer Build
   </message>
-  <message name="IDS_VERSION_UI_32BIT" desc="32-bit on the chrome://version page">
+  <message name="IDS_VERSION_UI_32BIT" desc="32-bit on the dao://version page">
     (32-bit)
   </message>
-  <message name="IDS_VERSION_UI_64BIT" desc="64-bit on the chrome://version page">
+  <message name="IDS_VERSION_UI_64BIT" desc="64-bit on the dao://version page">
     (64-bit)
   </message>
   <if expr="is_macosx">
-    <message name="IDS_VERSION_UI_64BIT_INTEL" desc="On the chrome://version page, an indicator that this is an instance of Chromium built for the 64-bit x86 processor. Translation is not needed; this is a technical term.">
+    <message name="IDS_VERSION_UI_64BIT_INTEL" desc="On the dao://version page, an indicator that this is an instance of Chromium built for the 64-bit x86 processor. Translation is not needed; this is a technical term.">
       (x86_64)
     </message>
-    <message name="IDS_VERSION_UI_64BIT_TRANSLATED_INTEL" desc="On the chrome://version page, an indicator that this is an instance of Chromium built for the 64-bit x86 processor, but being run on an Arm processor. Do not translate 'x86_64' as that is a technical term. The 'translated' in this string refers to the fact that these two processors use different languages, so the sense of 'translated' for this string is as in a language translation.">
+    <message name="IDS_VERSION_UI_64BIT_TRANSLATED_INTEL" desc="On the dao://version page, an indicator that this is an instance of Chromium built for the 64-bit x86 processor, but being run on an Arm processor. Do not translate 'x86_64' as that is a technical term. The 'translated' in this string refers to the fact that these two processors use different languages, so the sense of 'translated' for this string is as in a language translation.">
       (x86_64 translated)
     </message>
   </if>
   <if expr="is_macosx or is_win">
-    <message name="IDS_VERSION_UI_64BIT_ARM" desc="On the chrome://version page, an indicator that this is an instance of Chromium built for the 64-bit Arm processor. Translation is not needed; this is a technical term.">
+    <message name="IDS_VERSION_UI_64BIT_ARM" desc="On the dao://version page, an indicator that this is an instance of Chromium built for the 64-bit Arm processor. Translation is not needed; this is a technical term.">
       (arm64)
     </message>
   </if>
   <if expr="is_win">
-    <message name="IDS_VERSION_UI_32BIT_TRANSLATED_INTEL" desc="On the chrome://version page, an indicator that this is an instance of Chromium built for the 32-bit x86 processor, but being run on an Arm processor. Do not translate.">
+    <message name="IDS_VERSION_UI_32BIT_TRANSLATED_INTEL" desc="On the dao://version page, an indicator that this is an instance of Chromium built for the 32-bit x86 processor, but being run on an Arm processor. Do not translate.">
       (32-bit emulated)
     </message>
-    <message name="IDS_VERSION_UI_64BIT_TRANSLATED_INTEL" desc="On the chrome://version page, an indicator that this is an instance of Chromium built for the 64-bit x86 processor, but being run on an Arm processor. Do not translate.">
+    <message name="IDS_VERSION_UI_64BIT_TRANSLATED_INTEL" desc="On the dao://version page, an indicator that this is an instance of Chromium built for the 64-bit x86 processor, but being run on an Arm processor. Do not translate.">
       (64-bit emulated)
     </message>
   </if>
