diff --git a/chrome/browser/ui/webui/about/about_ui.cc b/chrome/browser/ui/webui/about/about_ui.cc
index 0939e62b45..ce0ab88d76 100644
--- a/chrome/browser/ui/webui/about/about_ui.cc
+++ b/chrome/browser/ui/webui/about/about_ui.cc
@@ -475,8 +475,8 @@ std::string ChromeURLs(content::BrowserContext* browser_context) {
 #if BUILDFLAG(ENABLE_SESSION_SERVICE)
   // Add the session service internals page to the end.
   html += base::StrCat(
-      {"<li><a href='chrome://internals/",
-       chrome::kChromeUISessionServiceInternalsPath, "'>chrome://internals/",
+      {"<li><a href='dao://internals/",
+       chrome::kChromeUISessionServiceInternalsPath, "'>dao://internals/",
        chrome::kChromeUISessionServiceInternalsPath, "</a></li>\n"});
 #endif  // BUILDFLAG(ENABLE_SESSION_SERVICE)
 
