diff --git a/chrome/browser/resources/settings/reset_page/reset_profile_dialog.ts b/chrome/browser/resources/settings/reset_page/reset_profile_dialog.ts
index 675a858b42..d79b0eaa9d 100644
--- a/chrome/browser/resources/settings/reset_page/reset_profile_dialog.ts
+++ b/chrome/browser/resources/settings/reset_page/reset_profile_dialog.ts
@@ -37,7 +37,6 @@ export interface SettingsResetProfileDialogElement {
     cancel: CrButtonElement,
     dialog: CrDialogElement,
     reset: CrButtonElement,
-    sendSettings: CrCheckboxElement,
   };
 }
 
@@ -115,8 +114,6 @@ export class SettingsResetProfileDialogElement extends
       this.browserProxy_.onHideResetProfileDialog();
     });
 
-    this.shadowRoot!.querySelector('cr-checkbox a')!.addEventListener(
-        'click', this.onShowReportedSettingsClick_.bind(this));
   }
 
   private showDialog_() {
@@ -156,7 +153,7 @@ export class SettingsResetProfileDialogElement extends
     this.clearingInProgress_ = true;
     this.browserProxy_
         .performResetProfileSettings(
-            this.$.sendSettings.checked, this.resetRequestOrigin_)
+            false, this.resetRequestOrigin_)
         .then(() => {
           this.clearingInProgress_ = false;
           if (this.$.dialog.open) {
@@ -167,13 +164,6 @@ export class SettingsResetProfileDialogElement extends
         });
   }
 
-  /**
-   * Displays the settings that will be reported in a new tab.
-   */
-  private onShowReportedSettingsClick_(e: Event) {
-    this.browserProxy_.showReportedSettings();
-    e.stopPropagation();
-  }
 }
 
 declare global {
