diff --git a/chrome/browser/ui/views/frame/browser_view.h b/chrome/browser/ui/views/frame/browser_view.h
index b9986c6e90..f80bd0e0ca 100644
--- a/chrome/browser/ui/views/frame/browser_view.h
+++ b/chrome/browser/ui/views/frame/browser_view.h
@@ -124,6 +124,14 @@ namespace glic {
 class BorderView;
 }  // namespace glic
 
+namespace dao {
+class DaoAddressBarView;
+class DaoCommandBarView;
+class DaoSidebarView;
+class DaoCornerOverlayView;
+class DaoDownloadFlyoutView;
+}  // namespace dao
+
 namespace segmentation_platform {
 struct ClassificationResult;
 }
@@ -247,6 +255,12 @@ class BrowserView : public BrowserWindow,
 
   SidePanel* unified_side_panel() { return unified_side_panel_; }
 
+  dao::DaoAddressBarView* dao_address_bar() { return dao_address_bar_; }
+  dao::DaoSidebarView* dao_sidebar() { return dao_sidebar_; }
+  dao::DaoCornerOverlayView* dao_corner_overlay() { return dao_corner_overlay_; }
+  dao::DaoCommandBarView* dao_command_bar() { return dao_command_bar_; }
+  dao::DaoDownloadFlyoutView* dao_download_flyout() { return dao_download_flyout_; }
+
   void set_contents_border_widget(views::Widget* contents_border_widget) {
     GetBrowserViewLayout()->set_contents_border_widget(contents_border_widget);
   }
@@ -1280,6 +1293,13 @@ class BrowserView : public BrowserWindow,
   raw_ptr<views::View> left_aligned_side_panel_separator_ = nullptr;
   raw_ptr<views::View> side_panel_rounded_corner_ = nullptr;
 
+  // Dao left sidebar, corner overlay, address bar, command bar, and download flyout
+  raw_ptr<dao::DaoAddressBarView> dao_address_bar_ = nullptr;
+  raw_ptr<dao::DaoSidebarView> dao_sidebar_ = nullptr;
+  raw_ptr<dao::DaoCornerOverlayView> dao_corner_overlay_ = nullptr;
+  raw_ptr<dao::DaoCommandBarView> dao_command_bar_ = nullptr;
+  raw_ptr<dao::DaoDownloadFlyoutView> dao_download_flyout_ = nullptr;
+
   // Provides access to the toolbar buttons this browser view uses. Buttons may
   // appear in a hosted app frame or in a tabbed UI toolbar.
   raw_ptr<ToolbarButtonProvider> toolbar_button_provider_ = nullptr;
