diff --git a/chrome/browser/ui/views/frame/contents_web_view.cc b/chrome/browser/ui/views/frame/contents_web_view.cc
index f5e8ba18cb..c138206ff7 100644
--- a/chrome/browser/ui/views/frame/contents_web_view.cc
+++ b/chrome/browser/ui/views/frame/contents_web_view.cc
@@ -58,6 +58,16 @@ void ContentsWebView::SetBackgroundVisible(bool background_visible) {
   }
 }
 
+void ContentsWebView::SetBackgroundOverrideColor(std::optional<SkColor> color) {
+  background_override_color_ = color;
+  if (GetWidget()) {
+    UpdateBackgroundColor();
+  } else if (layer() && background_override_color_.has_value()) {
+    layer()->SetColor(background_visible_ ? *background_override_color_
+                                          : SK_ColorTRANSPARENT);
+  }
+}
+
 const gfx::RoundedCornersF& ContentsWebView::GetBackgroundRadii() const {
   const ui::Layer* background_layer = layer();
 
@@ -119,9 +129,10 @@ void ContentsWebView::SetWebContents(content::WebContents* web_contents) {
 }
 
 void ContentsWebView::UpdateBackgroundColor() {
-  const SkColor color = GetColorProvider()->GetColor(
-      is_letterboxing() ? kColorWebContentsBackgroundLetterboxing
-                        : kColorWebContentsBackground);
+  const SkColor color = background_override_color_.value_or(
+      GetColorProvider()->GetColor(
+          is_letterboxing() ? kColorWebContentsBackgroundLetterboxing
+                            : kColorWebContentsBackground));
   // `color` must be opaque, see RenderWidgetHostView::SetBackgroundColor() for
   // details.
   // TODO(crbug.com/456309057): Update this to be a CHECK.
