diff --git a/chrome/browser/prefs/session_startup_pref.cc b/chrome/browser/prefs/session_startup_pref.cc
index 266dc5f675..3fe94ab156 100644
--- a/chrome/browser/prefs/session_startup_pref.cc
+++ b/chrome/browser/prefs/session_startup_pref.cc
@@ -66,11 +66,8 @@ void SessionStartupPref::RegisterProfilePrefs(
 
 // static
 SessionStartupPref::Type SessionStartupPref::GetDefaultStartupType() {
-#if BUILDFLAG(IS_CHROMEOS)
+  // Dao Browser: Always restore last session by default.
   return SessionStartupPref::LAST;
-#else
-  return SessionStartupPref::DEFAULT;
-#endif
 }
 
 // static
