diff --git a/chrome/browser/prefs/session_startup_pref.cc b/chrome/browser/prefs/session_startup_pref.cc
index dd071315ef..ff84801b2b 100644
--- a/chrome/browser/prefs/session_startup_pref.cc
+++ b/chrome/browser/prefs/session_startup_pref.cc
@@ -67,16 +67,8 @@ void SessionStartupPref::RegisterProfilePrefs(
 
 // static
 SessionStartupPref::Type SessionStartupPref::GetDefaultStartupType() {
-#if BUILDFLAG(IS_CHROMEOS)
+  // Dao Browser: Always restore last session by default.
   return SessionStartupPref::LAST;
-#else
-#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_MAC)
-  if (features::kSetDefaultToContinueSession.Get()) {
-    return SessionStartupPref::LAST;
-  }
-#endif
-  return SessionStartupPref::DEFAULT;
-#endif
 }
 
 // static
