diff --git a/chrome/browser/resources/print_preview/ui/plugin_proxy.ts b/chrome/browser/resources/print_preview/ui/plugin_proxy.ts
index afbf6c857e..fcdbe21516 100644
--- a/chrome/browser/resources/print_preview/ui/plugin_proxy.ts
+++ b/chrome/browser/resources/print_preview/ui/plugin_proxy.ts
@@ -85,7 +85,7 @@ export class PluginProxyImpl implements PluginProxy {
     assert(!this.plugin_);
     const srcUrl = this.getPreviewUrl_(previewUid, index);
     this.plugin_ = pdfCreateOutOfProcessPlugin(
-        srcUrl, 'chrome://print/pdf/index_print.html');
+        srcUrl, 'dao://print/pdf/index_print.html');
     this.plugin_.classList.add('preview-area-plugin');
     // NOTE: The plugin's 'id' field must be set to 'pdf-viewer' since
     // chrome/renderer/printing/print_render_frame_helper.cc actually
