diff --git a/content/public/common/url_constants.h b/content/public/common/url_constants.h
index 55ea2d2632..7bd7dc1f1f 100644
--- a/content/public/common/url_constants.h
+++ b/content/public/common/url_constants.h
@@ -18,7 +18,7 @@ namespace content {
 // from there.
 inline constexpr char kChromeDevToolsScheme[] = "devtools";
 inline constexpr char kChromeErrorScheme[] = "chrome-error";
-inline constexpr char kChromeUIScheme[] = "chrome";  // Used for WebUIs.
+inline constexpr char kChromeUIScheme[] = "dao";  // Used for WebUIs.
 inline constexpr char kChromeNativeScheme[] = "chrome-native";
 inline constexpr char kChromeUIUntrustedScheme[] = "chrome-untrusted";
 inline constexpr char kViewSourceScheme[] = "view-source";
