--- a/content/public/common/url_constants.h
+++ b/content/public/common/url_constants.h
@@ -18,7 +18,7 @@ namespace content {
 // TODO(jam): some of these don't below in the content layer, but are accessed
 // from there.
 inline constexpr char kChromeDevToolsScheme[] = "devtools";
 inline constexpr char kChromeErrorScheme[] = "chrome-error";
-inline constexpr char kChromeUIScheme[] = "chrome";  // Used for WebUIs.
+inline constexpr char kChromeUIScheme[] = "dao";  // Used for WebUIs.
 inline constexpr char kChromeNativeScheme[] = "chrome-native";
 inline constexpr char kChromeUIUntrustedScheme[] = "chrome-untrusted";
