diff --git a/components/download/public/common/download_item.cc b/components/download/public/common/download_item.cc
--- a/components/download/public/common/download_item.cc
+++ b/components/download/public/common/download_item.cc
@@ -6,11 +6,22 @@
 
 #include "base/check.h"
 #include "base/notreached.h"
+#include "components/download/internal/common/parallel_download_utils.h"
 
 namespace download {
 
 DownloadItem::Observer::~Observer() {
   CHECK(!IsInObserverList());
+}
+
+void DownloadItem::SetParallelDownloadEnabled(bool enabled) {
+  if (!parallel_download_enabled_.has_value()) {
+    parallel_download_enabled_ = enabled;
+  }
+}
+
+bool DownloadItem::IsParallelDownloadEnabledForItem() const {
+  return parallel_download_enabled_.value_or(IsParallelDownloadEnabled());
 }
 
 void DownloadItem::SetStateForTesting(DownloadState state) {
