diff --git a/chrome/app/chrome_command_ids.h b/chrome/app/chrome_command_ids.h
index 1384557ac6..8b70fcfa76 100644
--- a/chrome/app/chrome_command_ids.h
+++ b/chrome/app/chrome_command_ids.h
@@ -86,6 +86,12 @@
 #define IDC_TOGGLE_VERTICAL_TABS        34056
 #define IDC_VERTICAL_TABS_SEND_FEEDBACK 34057
 
+// Dao commands
+#define IDC_NEW_LITTLE_DAO              34070
+#define IDC_DAO_COPY_URL                34071
+#define IDC_DAO_DUPLICATE_TAB           34072
+#define IDC_DAO_WELCOME                 34073
+
 // Web app window commands
 #define IDC_COPY_URL                    34060
 #define IDC_OPEN_IN_CHROME              34061
