diff --git a/chrome/app/generated_resources.grd b/chrome/app/generated_resources.grd
index c25aadab11..3df8006458 100644
--- a/chrome/app/generated_resources.grd
+++ b/chrome/app/generated_resources.grd
@@ -921,7 +921,7 @@ are declared in tools/grit/grit_args.gni.
           <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_SELECT_PASSWORD" desc="This is a context menu entry in the passwords submenu. It triggers the password selection bubble.">
             Select password
           </message>
-          <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_IMPORT_PASSWORDS" desc="This is a context menu entry. It opens chrome://password-manager. It can be either in the context menu (for non-syncing users) or in the passwords submenu (for syncing users).">
+          <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_IMPORT_PASSWORDS" desc="This is a context menu entry. It opens dao://password-manager. It can be either in the context menu (for non-syncing users) or in the passwords submenu (for syncing users).">
             Import passwords
           </message>
           <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_SUGGEST_PASSWORD" desc="This is a context menu entry in the passwords submenu. It triggers password generation.">
@@ -1259,7 +1259,7 @@ are declared in tools/grit/grit_args.gni.
           <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_SELECT_PASSWORD" desc="In Title Case: This is a context menu entry in the passwords submenu. It triggers the password selection bubble.">
             Select Password
           </message>
-          <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_IMPORT_PASSWORDS" desc="In Title Case: This is a context menu entry. It opens chrome://password-manager. It can be either in the context menu (for non-syncing users) or in the passwords submenu (for syncing users).">
+          <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_IMPORT_PASSWORDS" desc="In Title Case: This is a context menu entry. It opens dao://password-manager. It can be either in the context menu (for non-syncing users) or in the passwords submenu (for syncing users).">
             Import Passwords
           </message>
           <message name="IDS_CONTENT_CONTEXT_AUTOFILL_FALLBACK_PASSWORDS_SUGGEST_PASSWORD" desc="In Title Case: This is a context menu entry in the passwords submenu. It triggers password generation.">
@@ -1811,7 +1811,7 @@ are declared in tools/grit/grit_args.gni.
         <message name="IDS_EXTENSIONS_SUBMENU" desc="The extensions menu in the app menu (title case).">
           &amp;Extensions
         </message>
-        <message name="IDS_EXTENSIONS_SUBMENU_MANAGE_EXTENSIONS_ITEM" desc="The text for the link in the extensions menu, in the app menu, to visit chrome://extensions to view all extensions installed (title case).">
+        <message name="IDS_EXTENSIONS_SUBMENU_MANAGE_EXTENSIONS_ITEM" desc="The text for the link in the extensions menu, in the app menu, to visit dao://extensions to view all extensions installed (title case).">
           Manage Extensions
         </message>
         <message name="IDS_EXTENSIONS_SUBMENU_CHROME_WEBSTORE_ITEM" desc="The text for the link in the extensions menu, in the app menu, to visit the Chrome Web Store to get more extensions (title case).">
@@ -1870,7 +1870,7 @@ are declared in tools/grit/grit_args.gni.
         <message name="IDS_EXTENSIONS_SUBMENU" desc="In Title Case: The extensions menu in the app menu.">
           &amp;Extensions
         </message>
-        <message name="IDS_EXTENSIONS_SUBMENU_MANAGE_EXTENSIONS_ITEM" desc="In Title Case: The text for the link in the extensions menu, in the app menu, to visit chrome://extensions to view all extensions installed.">
+        <message name="IDS_EXTENSIONS_SUBMENU_MANAGE_EXTENSIONS_ITEM" desc="In Title Case: The text for the link in the extensions menu, in the app menu, to visit dao://extensions to view all extensions installed.">
           Manage Extensions
         </message>
         <message name="IDS_EXTENSIONS_SUBMENU_CHROME_WEBSTORE_ITEM" desc="In Title Case: The text for the link in the extensions menu, in the app menu, to visit the Chrome Web Store to get more extensions.">
@@ -2482,7 +2482,7 @@ are declared in tools/grit/grit_args.gni.
                desc="Placeholder text/label for the search input in the download history page">
         Search download history
       </message>
-      <message name="IDS_DOWNLOAD_NO_DOWNLOADS" desc="A message shown when the user has no downloads to show on chrome://downloads.">
+      <message name="IDS_DOWNLOAD_NO_DOWNLOADS" desc="A message shown when the user has no downloads to show on dao://downloads.">
         Files you download appear here
       </message>
       <message name="IDS_DOWNLOAD_LINK_RESUME"
@@ -5991,7 +5991,7 @@ are declared in tools/grit/grit_args.gni.
         </message>
       </if>
 
-      <!-- Extensions strings, which are not used by chrome://extensions. -->
+      <!-- Extensions strings, which are not used by dao://extensions. -->
       <if expr="enable_extensions_core">
         <message name="IDS_EXTENSIONS_WANTS_ACCESS_TO_SITE" desc="The message shown on mouseover when an extension wants to run on a specific site.">
           Wants access to this site
@@ -6025,7 +6025,7 @@ are declared in tools/grit/grit_args.gni.
           <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_RUN_ON_ALL_SITES_V2" desc="The label in an extension's context menu to always allow the extension access on all sites (sentence case).">
             Always on all sites
           </message>
-           <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_PERMISSIONS_PAGE" desc="The label in an extension's context menu that the user can click to open site permissions page in chrome://extensions (sentence case).">
+           <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_PERMISSIONS_PAGE" desc="The label in an extension's context menu that the user can click to open site permissions page in dao://extensions (sentence case).">
             Manage site permissions
           </message>
           <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_RUN_ON_CLICK" desc="The label in an extension's context menu to allow the extension access to the page only when the user clicks on the extension action (sentence case).">
@@ -6099,7 +6099,7 @@ are declared in tools/grit/grit_args.gni.
           <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_RUN_ON_ALL_SITES_V2" desc="The label in an extension's context menu to always allow the extension access on all sites (title case).">
             Always on All Sites
           </message>
-          <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_PERMISSIONS_PAGE" desc="The label in an extension's context menu that the user can click to open site permissions page in chrome://extensions (title case).">
+          <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_PERMISSIONS_PAGE" desc="The label in an extension's context menu that the user can click to open site permissions page in dao://extensions (title case).">
             Manage Site Permissions
           </message>
           <message name="IDS_EXTENSIONS_CONTEXT_MENU_PAGE_ACCESS_RUN_ON_CLICK" desc="The label in an extension's context menu to allow the extension access to the page only when the user clicks on the extension action (title case).">
@@ -6135,13 +6135,13 @@ are declared in tools/grit/grit_args.gni.
           <message name="IDS_EXTENSIONS_PINNED_BY_ADMIN" desc="The text indicating that an extension has been 'pinned' to the toolbar by the controlling administrator (title case).">
             Pinned by your Administrator
           </message>
-          <message name="IDS_MANAGE_EXTENSION" desc="The text for the button in the extension's context menu used to open the chrome://extensions/?id=xxx page and manage a specific extension (title case).">
+          <message name="IDS_MANAGE_EXTENSION" desc="The text for the button in the extension's context menu used to open the dao://extensions/?id=xxx page and manage a specific extension (title case).">
             Manage Extension
           </message>
-          <message name="IDS_MANAGE_EXTENSIONS" desc="The text for the button in the extensions toolbar menu used to open the chrome://extensions page and manage all extensions (title case).">
+          <message name="IDS_MANAGE_EXTENSIONS" desc="The text for the button in the extensions toolbar menu used to open the dao://extensions page and manage all extensions (title case).">
             Manage Extensions
           </message>
-          <message name="IDS_VIEW_WEB_PERMISSIONS" desc="The text for the button in the extension context menu used to open the chrome://settings/content/siteDetails?site=xxx page and view the site details for a specific extension (title case).">
+          <message name="IDS_VIEW_WEB_PERMISSIONS" desc="The text for the button in the extension context menu used to open the dao://settings/content/siteDetails?site=xxx page and view the site details for a specific extension (title case).">
             View Web Permissions
           </message>
           <message name="IDS_EXTENSION_ACTION_INSPECT_POPUP" desc="The text for the right-click menu of page and browser actions which shows the popup and opens the developer tools (title case).">
@@ -6602,7 +6602,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
       <message name="IDS_DEPRECATED_APPS_LEARN_MORE_AX_LABEL" desc="Accessibility label text for IDS_DEPRECATED_APPS_LEARN_MORE link">
         Learn more about unsupported Chrome apps
       </message>
-      <message name="IDS_DEPRECATED_APPS_DELETION_LINK" desc="Contains link to trigger the deprecated apps deletion dialog from chrome://apps">
+      <message name="IDS_DEPRECATED_APPS_DELETION_LINK" desc="Contains link to trigger the deprecated apps deletion dialog from dao://apps">
         {NUM_APPS, plural,
         =1 {Remove 1 unsupported app}
         other {Remove # unsupported apps}}
@@ -7016,7 +7016,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
           </message>
         </then>
         <else>
-          <message name="IDS_WEB_APP_SETTINGS_LINK" desc="Text of the link pointing to the app settings page. The link appears in app window/tab -> page info bubble in place of Site Settings, and in chrome://apps right click menu">
+          <message name="IDS_WEB_APP_SETTINGS_LINK" desc="Text of the link pointing to the app settings page. The link appears in app window/tab -> page info bubble in place of Site Settings, and in dao://apps right click menu">
             Settings
           </message>
           <message name="IDS_WEB_APP_SETTINGS_LINK_TOOLTIP" desc="The text of the tooltip on IDS_WEB_APP_SETTINGS_LINK.">
@@ -7026,7 +7026,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
       </if>
 
       <!-- Components -->
-      <message name="IDS_COMPONENTS_TITLE" desc="Title for the chrome://components page.">
+      <message name="IDS_COMPONENTS_TITLE" desc="Title for the dao://components page.">
         Components
       </message>
       <message name="IDS_COMPONENTS_VERSION" desc="The label in front of a component version number.">
@@ -7273,52 +7273,52 @@ Keep your key file in a safe place. You will need it to create new versions of y
       <!-- End of Sync Promo strings for Desktop Identity Consistency. -->
 
       <!-- WebRTC logs -->
-      <message name="IDS_WEBRTC_LOGS_TITLE" desc="Title for the chrome://webrtc-logs page.">
+      <message name="IDS_WEBRTC_LOGS_TITLE" desc="Title for the dao://webrtc-logs page.">
         WebRTC logs
       </message>
-      <message name="IDS_WEBRTC_TEXT_LOGS_LOG_COUNT_BANNER_FORMAT" desc="Format for WebRTC text log count banner on chrome://webrtc-logs">
+      <message name="IDS_WEBRTC_TEXT_LOGS_LOG_COUNT_BANNER_FORMAT" desc="Format for WebRTC text log count banner on dao://webrtc-logs">
         WebRTC text logs (<ph name="WEBRTC_TEXT_LOG_COUNT">$1<ex>3</ex></ph>)
       </message>
-      <message name="IDS_WEBRTC_EVENT_LOGS_LOG_COUNT_BANNER_FORMAT" desc="Format for WebRTC event log count banner on chrome://webrtc-logs">
+      <message name="IDS_WEBRTC_EVENT_LOGS_LOG_COUNT_BANNER_FORMAT" desc="Format for WebRTC event log count banner on dao://webrtc-logs">
         WebRTC event logs (<ph name="WEBRTC_EVENT_LOG_COUNT">$1<ex>3</ex></ph>)
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_HEADER_FORMAT" desc="Format for uploaded log entry headings on chrome://webrtc-logs">
+      <message name="IDS_WEBRTC_LOGS_LOG_HEADER_FORMAT" desc="Format for uploaded log entry headings on dao://webrtc-logs">
         WebRTC log captured <ph name="WEBRTC_LOG_CAPTURE_TIME">$1<ex>Tuesday, January 25, 2011 2:58:02 PM</ex></ph>
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_LOCAL_FILE_LABEL_FORMAT" desc="Format for local log entry label on chrome://webrtc-logs">
+      <message name="IDS_WEBRTC_LOGS_LOG_LOCAL_FILE_LABEL_FORMAT" desc="Format for local log entry label on dao://webrtc-logs">
         Local file:
       </message>
-      <message name="IDS_WEBRTC_LOGS_NO_LOCAL_LOG_FILE_MESSAGE" desc="The explanatory message for chrome://webrtc-logs when there is no local log file">
+      <message name="IDS_WEBRTC_LOGS_NO_LOCAL_LOG_FILE_MESSAGE" desc="The explanatory message for dao://webrtc-logs when there is no local log file">
         No local log file.
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_UPLOAD_TIME_FORMAT" desc="Format for uploaded log entry upload time on chrome://webrtc-logs">
+      <message name="IDS_WEBRTC_LOGS_LOG_UPLOAD_TIME_FORMAT" desc="Format for uploaded log entry upload time on dao://webrtc-logs">
         Uploaded <ph name="WEBRTC_LOG_UPLOAD_TIME">$1<ex>Tuesday, January 25, 2011 2:58:02 PM</ex></ph>
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_FAILED_UPLOAD_TIME_FORMAT" desc="Format for a log's failed upload attempt entry on chrome://webrtc-logs">
+      <message name="IDS_WEBRTC_LOGS_LOG_FAILED_UPLOAD_TIME_FORMAT" desc="Format for a log's failed upload attempt entry on dao://webrtc-logs">
         Upload failed (<ph name="WEBRTC_LOG_UPLOAD_TIME">$1<ex>Tuesday, January 25, 2011 2:58:02 PM</ex></ph>).
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_REPORT_ID_FORMAT" desc="Format for uploaded log entry report ID on chrome://webrtc-logs. This is the ID on the server to which the log was uploaded.">
+      <message name="IDS_WEBRTC_LOGS_LOG_REPORT_ID_FORMAT" desc="Format for uploaded log entry report ID on dao://webrtc-logs. This is the ID on the server to which the log was uploaded.">
         Report ID <ph name="WEBRTC_LOG_REPORT_ID">$1<ex>8fa95dbb6f2ec862</ex></ph>
       </message>
-      <message name="IDS_WEBRTC_LOGS_BUG_LINK_LABEL" desc="Link text for filing a bug on chrome://webrtc-logs">
+      <message name="IDS_WEBRTC_LOGS_BUG_LINK_LABEL" desc="Link text for filing a bug on dao://webrtc-logs">
         File bug
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_PENDING_MESSAGE" desc="The explanatory message for chrome://webrtc-logs when the log is still pending upload.">
+      <message name="IDS_WEBRTC_LOGS_LOG_PENDING_MESSAGE" desc="The explanatory message for dao://webrtc-logs when the log is still pending upload.">
         Log upload pending.
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_ACTIVELY_UPLOADED_MESSAGE" desc="The explanatory message for chrome://webrtc-logs when the log is currently being uploaded.">
+      <message name="IDS_WEBRTC_LOGS_LOG_ACTIVELY_UPLOADED_MESSAGE" desc="The explanatory message for dao://webrtc-logs when the log is currently being uploaded.">
         Log upload underway.
       </message>
-      <message name="IDS_WEBRTC_LOGS_LOG_NOT_UPLOADED_MESSAGE" desc="The explanatory message for chrome://webrtc-logs when the log has not been uploaded">
+      <message name="IDS_WEBRTC_LOGS_LOG_NOT_UPLOADED_MESSAGE" desc="The explanatory message for dao://webrtc-logs when the log has not been uploaded">
         Log not uploaded.
       </message>
       <message name="IDS_WEBRTC_LOGS_EVENT_LOG_LOCAL_LOG_ID" desc="The ID of the log. This ID is assigned even if the log is not uploaded.">
         Local log ID: <ph name="WEBRTC_EVENT_LOG_LOCAL_ID">$1<ex>A83BC19927509874309F2EE002174235</ex></ph>.
       </message>
-      <message name="IDS_WEBRTC_LOGS_NO_TEXT_LOGS_MESSAGE" desc="The explanatory message for chrome://webrtc-logs when there are no captured text logs">
+      <message name="IDS_WEBRTC_LOGS_NO_TEXT_LOGS_MESSAGE" desc="The explanatory message for dao://webrtc-logs when there are no captured text logs">
         You have no recently captured WebRTC text logs.
       </message>
-      <message name="IDS_WEBRTC_LOGS_NO_EVENT_LOGS_MESSAGE" desc="The explanatory message for chrome://webrtc-logs when there are no captured event logs">
+      <message name="IDS_WEBRTC_LOGS_NO_EVENT_LOGS_MESSAGE" desc="The explanatory message for dao://webrtc-logs when there are no captured event logs">
         You have no recently captured WebRTC event logs.
       </message>
 
@@ -7445,7 +7445,7 @@ Keep your key file in a safe place. You will need it to create new versions of y
         </message>
       </if>
 
-      <!-- chrome://browser-switch strings -->
+      <!-- dao://browser-switch strings -->
       <if expr="toolkit_views and not is_chromeos">
         <message name="IDS_ABOUT_BROWSER_SWITCH_TITLE" desc="about:browser-switch page title">
           Legacy Browser Support
@@ -7478,14 +7478,14 @@ Keep your key file in a safe place. You will need it to create new versions of y
           Only http, https and file protocols are supported for browser redirects.
         </message>
 
-        <!-- chrome://browser-switch/internals strings -->
+        <!-- dao://browser-switch/internals strings -->
         <message name="IDS_ABOUT_BROWSER_SWITCH_INTERNALS_DESC" desc="about:browser-switch/internals page description">
           Legacy Browser Support (LBS) allows specific URLs patterns to be opened in an alternative browser that supports legacy features required to properly run those sites.
         </message>
         <message name="IDS_ABOUT_BROWSER_SWITCH_INTERNALS_TITLE" desc="about:browser-switch/internals page title">
           Legacy Browser Support (LBS) - Internals
         </message>
-        <message name="IDS_ABOUT_BROWSER_SWITCH_INTERNALS_NOTHING_SHOWN" desc="nothing to show on chrome://browser-switch/internals, because the policy BrowserSwitcherEnabled is disabled">
+        <message name="IDS_ABOUT_BROWSER_SWITCH_INTERNALS_NOTHING_SHOWN" desc="nothing to show on dao://browser-switch/internals, because the policy BrowserSwitcherEnabled is disabled">
           Nothing to show
         </message>
         <message name="IDS_ABOUT_BROWSER_SWITCH_INTERNALS_LBS_DISABLED" desc="about:browser-switch/internals text shown when the policy BrowserSwitcherEnabled is disabled">
@@ -9279,13 +9279,13 @@ Keep your key file in a safe place. You will need it to create new versions of y
 
       <!-- Extensions NTP Middle Slot Promo -->
       <message name="IDS_EXTENSIONS_PROMO_PERFORMANCE">
-        Web browsing should be fast. Take a moment to <ph name="BEGIN_LINK">&lt;a href="chrome://extensions"&gt;</ph>check your extensions<ph name="END_LINK">&lt;/a&gt;</ph> now.
+        Web browsing should be fast. Take a moment to <ph name="BEGIN_LINK">&lt;a href="dao://extensions"&gt;</ph>check your extensions<ph name="END_LINK">&lt;/a&gt;</ph> now.
       </message>
       <message name="IDS_EXTENSIONS_PROMO_PRIVACY">
-        Safety matters. Take a moment and <ph name="BEGIN_LINK">&lt;a href="chrome://extensions"&gt;</ph>check your extensions now<ph name="END_LINK">&lt;/a&gt;</ph>
+        Safety matters. Take a moment and <ph name="BEGIN_LINK">&lt;a href="dao://extensions"&gt;</ph>check your extensions now<ph name="END_LINK">&lt;/a&gt;</ph>
       </message>
       <message name="IDS_EXTENSIONS_PROMO_NEUTRAL">
-        Got extensions? <ph name="BEGIN_LINK">&lt;a href="chrome://extensions"&gt;</ph>Manage your extensions<ph name="END_LINK">&lt;/a&gt;</ph> in one easy place.
+        Got extensions? <ph name="BEGIN_LINK">&lt;a href="dao://extensions"&gt;</ph>Manage your extensions<ph name="END_LINK">&lt;/a&gt;</ph> in one easy place.
       </message>
 
       <!-- New Tab Footer -->
@@ -13812,7 +13812,7 @@ Check your passwords anytime in <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Google
         Set later
       </message>
 
-      <!-- chrome://help -->
+      <!-- dao://help -->
       <if expr="not is_android">
         <if expr="not is_chromeos and use_titlecase">
           <message name="IDS_RELAUNCH_BUTTON" desc="Titlecase: The label for the relaunch button that relaunches the browser once update is complete">
@@ -13850,7 +13850,7 @@ Check your passwords anytime in <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Google
         </message>
       </if> <!-- is_chromeos -->
 
-      <!-- Settings related strings not specific to chrome://settings.
+      <!-- Settings related strings not specific to dao://settings.
            (Settings specific strings are in settings_strings.grdp).
            Not necessary on Android that uses native UI in settings unless extensions are enabled. -->
       <if expr="not is_android or enable_extensions_core">
@@ -13913,7 +13913,7 @@ Check your passwords anytime in <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Google
         </message>
 
         <!-- Enhanced Safe Browsing CRX allowlist warning -->
-        <message name="IDS_EXTENSIONS_SAFE_BROWSING_CRX_ALLOWLIST_WARNING" desc="A warning shown in 'chrome://extensions' when the extension is not included in the Enhanced Safe Browsing CRX allowlist.">
+        <message name="IDS_EXTENSIONS_SAFE_BROWSING_CRX_ALLOWLIST_WARNING" desc="A warning shown in 'dao://extensions' when the extension is not included in the Enhanced Safe Browsing CRX allowlist.">
            This extension is not trusted by Enhanced Safe Browsing.
         </message>
         <message name="IDS_EXTENSIONS_SAFE_BROWSING_CRX_ALLOWLIST_WARNING_LEARN_MORE" desc="Text read by screen readers for a link labeled 'Learn more' that links to a Help article about Enhanced Safe Browsing.">
@@ -14580,7 +14580,7 @@ Check your passwords anytime in <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Google
         Your account <ph name="EMAIL">$1<ex>john.doe@example.com</ex></ph> is no longer allowed as the primary account. Because this account is managed by <ph name="DOMAIN">$2<ex>example.com</ex></ph>, your bookmarks, history, passwords, and other settings will be cleared from this device.
       </message>
 
-      <!-- Managed UI on pages like chrome://settings and chrome://downloads -->
+      <!-- Managed UI on pages like dao://settings and dao://downloads -->
       <if expr="not is_android">
         <if expr="is_chromeos">
           <message name="IDS_DEVICE_MANAGED_WITH_HYPERLINK" desc="Message to end users in Enterprise/EDU, with a link for more info (ChromeOS)">
@@ -17305,14 +17305,14 @@ Please help our engineers fix this problem. Tell us what happened right before y
       </message>
 
       <!-- About device logs UI display strings -->
-      <message name="IDS_DEVICE_LOG_TITLE" desc="Title of the chrome://device-log page">
+      <message name="IDS_DEVICE_LOG_TITLE" desc="Title of the dao://device-log page">
         Device Log
       </message>
       <message name="IDS_DEVICE_AUTO_REFRESH" desc="Instructions for how to auto-refresh the page">
-        Add a query param in URL to auto-refresh the page: chrome://device-log/?refresh=&lt;sec&gt;
+        Add a query param in URL to auto-refresh the page: dao://device-log/?refresh=&lt;sec&gt;
       </message>
       <message name="IDS_DEVICE_SELECT_TYPES" desc="Instructions for how to select types for the page on page load">
-        Add a query param in the URL to select types on page load. Example: chrome://device-log/?types=Bluetooth,USB
+        Add a query param in the URL to select types on page load. Example: dao://device-log/?types=Bluetooth,USB
       </message>
       <message name="IDS_DEVICE_LOG_REFRESH" desc="Button to refresh device log entries">
         Refresh
@@ -17392,7 +17392,7 @@ Please help our engineers fix this problem. Tell us what happened right before y
       <message name="IDS_DEVICE_LOG_TIME_DETAIL" desc="Detailed timestamps checkbox in device event log">
         Detailed Timestamps
       </message>
-      <message name="IDS_DEVICE_LOG_ENTRY" desc="Format string for lines in chrome://device-log.">
+      <message name="IDS_DEVICE_LOG_ENTRY" desc="Format string for lines in dao://device-log.">
         [<ph name="TIMESTAMP">$1<ex>Timestamp</ex></ph>]
         <ph name="FILE_INFO">$2<ex>file:123</ex></ph>
         <ph name="EVENT_NAME">$3<ex>Event Description</ex></ph>
@@ -19299,7 +19299,7 @@ Please help our engineers fix this problem. Tell us what happened right before y
       <message name="IDS_MEMORY_SAVER_DIALOG_TITLE" desc="Title text for a dialog with information about Memory Saver mode.">
         Memory Saver
       </message>
-      <message name="IDS_MEMORY_SAVER_DIALOG_SETTINGS_BUTTON" desc="Label for a button on a dialog that opens the chrome://settings/performance page when clicked.">
+      <message name="IDS_MEMORY_SAVER_DIALOG_SETTINGS_BUTTON" desc="Label for a button on a dialog that opens the dao://settings/performance page when clicked.">
         Settings
       </message>
       <message name="IDS_MEMORY_SAVER_DIALOG_SAVINGS_ACCNAME" desc="Accessible label for the memory savings shown on the Memory Saver dialog to provide context to the raw number.">
