diff --git a/chrome/app/google_chrome_strings.grd b/chrome/app/google_chrome_strings.grd
index 932be2d0ff..3e6b820e62 100644
--- a/chrome/app/google_chrome_strings.grd
+++ b/chrome/app/google_chrome_strings.grd
@@ -961,7 +961,7 @@ Google Chrome is unable to recover your settings.
         This extension hasn't published privacy practices, such as how it collects and uses data. Chrome recommends that you remove it.
       </message>
 
-      <!-- chrome://extensions page -->
+      <!-- dao://extensions page -->
     <message name="IDS_EXTENSIONS_SC_DESCRIPTION" desc="Detailed message displayed in the Safety Check description section.">
       {NUM_EXTENSIONS, plural,
       =1 {Chrome recommends that you remove it}
