diff --git a/chrome/app/password_manager_ui_strings.grdp b/chrome/app/password_manager_ui_strings.grdp
index d799fd1d13..affb72c5ed 100644
--- a/chrome/app/password_manager_ui_strings.grdp
+++ b/chrome/app/password_manager_ui_strings.grdp
@@ -586,8 +586,8 @@
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_MOVE_PASSWORDS_TO_ACCOUNT" desc="Description for the password section entry used for managing passwords.">
     {COUNT, plural,
-      =1 {{COUNT} password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='chrome://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}
-      other {{COUNT} passwords are saved only to this device. To use them on your other devices, <ph name="BEGIN_LINK">&lt;a href='chrome://password-manager/'&gt;</ph>save them in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}}
+      =1 {{COUNT} password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='dao://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}
+      other {{COUNT} passwords are saved only to this device. To use them on your other devices, <ph name="BEGIN_LINK">&lt;a href='dao://password-manager/'&gt;</ph>save them in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}}
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_MOVE_PASSWORDS_TO_ACCOUNT_DIALOG_DESCRIPTION" desc="Description for the dialog for uploading passwords to account storage.">
     {COUNT, plural,
@@ -853,7 +853,7 @@
       Sign in with <ph name="CREDENTIAL_PROVIDER"><ex>Google</ex>$1</ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_MOVE_SINGLE_PASSWORD_TO_ACCOUNT" desc="Description for the password details entry used for moving single password.">
-    This password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='chrome://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.
+    This password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='dao://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_MOVE_SINGLE_PASSWORD_TITLE" desc="Title for the move password dialog.">
     Use this password on all your devices
