diff --git a/chrome/app/password_manager_ui_strings.grdp b/chrome/app/password_manager_ui_strings.grdp
index 48b4a3e92a..279b85be31 100644
--- a/chrome/app/password_manager_ui_strings.grdp
+++ b/chrome/app/password_manager_ui_strings.grdp
@@ -571,8 +571,8 @@
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_MOVE_PASSWORDS_TO_ACCOUNT" desc="Description for the password section entry used for managing passwords.">
     {COUNT, plural,
-      =1 {{COUNT} password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='chrome://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}
-      other {{COUNT} passwords are saved only to this device. To use them on your other devices, <ph name="BEGIN_LINK">&lt;a href='chrome://password-manager/'&gt;</ph>save them in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}}
+      =1 {{COUNT} password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='dao://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}
+      other {{COUNT} passwords are saved only to this device. To use them on your other devices, <ph name="BEGIN_LINK">&lt;a href='dao://password-manager/'&gt;</ph>save them in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.}}
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_OPENS_IN_NEW_TAB" desc="ARIA (accessibility) label describing a link which opens in a new tab.">
     Opens in new tab
@@ -807,7 +807,7 @@
       Sign in with <ph name="CREDENTIAL_PROVIDER"><ex>Google</ex>$1</ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_MOVE_SINGLE_PASSWORD_TO_ACCOUNT" desc="Description for the password details entry used for moving single password.">
-    This password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='chrome://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.
+    This password is saved only to this device. To use it on your other devices, <ph name="BEGIN_LINK">&lt;a href='dao://password-manager/'&gt;</ph>save it in your Google Account<ph name="END_LINK">&lt;/a&gt;</ph>.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_MOVE_SINGLE_PASSWORD_TITLE" desc="Title for the move password dialog.">
     Use this password on all your devices
