diff --git a/chrome/app/support_tool_strings.grdp b/chrome/app/support_tool_strings.grdp
index 156b0ed66e..776a6d118a 100644
--- a/chrome/app/support_tool_strings.grdp
+++ b/chrome/app/support_tool_strings.grdp
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Support Tool UI (chrome://support-tool) strings (included from generated_resources.grd). -->
+<!-- Support Tool UI (dao://support-tool) strings (included from generated_resources.grd). -->
 <grit-part>
     <if expr="not is_android">
         <message name="IDS_SUPPORT_TOOL_ISSUE_DETAILS_PAGE_TITLE" desc="The title of first page (out of four pages) of Support Tool. This page contains details about the issue that the user wants to collect diagnose data. The title describes that the user needs to describe the issue on those fields.">
