diff --git a/chrome/browser/ui/views/overlay/back_to_tab_button.cc b/chrome/browser/ui/views/overlay/back_to_tab_button.cc
index 9c497749c6..ca0d392ff5 100644
--- a/chrome/browser/ui/views/overlay/back_to_tab_button.cc
+++ b/chrome/browser/ui/views/overlay/back_to_tab_button.cc
@@ -16,7 +16,7 @@
 namespace {
 
 constexpr int kBackToTabButtonVerticalMargin = 5;
-constexpr int kBackToTabButtonHorizontalMargin = 28;
+constexpr int kBackToTabButtonHorizontalMargin = 8;
 constexpr int kBackToTabButtonSize = 24;
 constexpr int kBackToTabButtonIconSize = 16;
 
@@ -41,7 +41,7 @@ OverlayWindowBackToTabButton::OverlayWindowBackToTabButton(
 
 void OverlayWindowBackToTabButton::SetPosition(const gfx::Size& size) {
   views::ImageButton::SetPosition(gfx::Point(
-      size.width() - kBackToTabButtonSize - kBackToTabButtonHorizontalMargin,
+      kBackToTabButtonHorizontalMargin,
       kBackToTabButtonVerticalMargin));
 }
 
