diff --git a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
index d1bfeca485..8e5f0ad843 100644
--- a/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
+++ b/chrome/browser/ui/views/overlay/video_overlay_window_views.cc
@@ -1443,7 +1443,7 @@ void VideoOverlayWindowViews::OnUpdateControlsBounds() {
   // The 2024 updated UI lays out controls differently.
   if (Use2024UI()) {
     constexpr int kTopControlsHeight = 34;
-    constexpr int kBottomControlsHeight = 64;
+    constexpr int kBottomControlsHeight = 36;
     constexpr int kTopScrimHeight = 160;
     constexpr int kBottomScrimHeight = 160;
     constexpr int kFaviconLeftMargin = 8;
@@ -1492,6 +1492,7 @@ void VideoOverlayWindowViews::OnUpdateControlsBounds() {
                                    top_controls_bounds.y() + kFaviconTopMargin},
                                   kFaviconSize);
     favicon_view_->SetPosition(favicon_view_bounds.origin());
+    favicon_view_->SetVisible(false);
 
     gfx::Point origin_position(
         favicon_view_bounds.right() + kFaviconRightMargin, kOriginTopMargin);
@@ -1499,6 +1500,7 @@ void VideoOverlayWindowViews::OnUpdateControlsBounds() {
     origin_->SetSize(
         {top_controls_bounds.width() - origin_position.x() - kOriginRightMargin,
          kOriginHeight});
+    origin_->SetVisible(false);
 
     minimize_button_->SetPosition(GetBounds().size(), quadrant);
     back_to_tab_button_->SetPosition(GetBounds().size(), quadrant);
@@ -1536,15 +1538,11 @@ void VideoOverlayWindowViews::OnUpdateControlsBounds() {
     // previous and next buttons. Here we calculate how much horizontal space
     // one of those buttons takes up and use that to calculate the width and x
     // position of the progress view.
-    constexpr int kPreviousNextTrackWidthPlusHorizontalMargins =
-        kBottomControlsHorizontalMargin + (2 * kControlHorizontalMargin) +
-        kActionButtonSize.width();
     progress_view_->SetPosition(
-        {bottom_controls_bounds.x() +
-             kPreviousNextTrackWidthPlusHorizontalMargins,
+        {bottom_controls_bounds.x() + kBottomControlsHorizontalMargin,
          bottom_controls_bounds.y() + kBottomControlsVerticalMargin});
     progress_view_->SetSize(
-        {bounds.width() - (2 * kPreviousNextTrackWidthPlusHorizontalMargins),
+        {bounds.width() - (2 * kBottomControlsHorizontalMargin),
          kProgressBarHeight});
 
     timestamp_->SetPosition(
@@ -1562,13 +1560,16 @@ void VideoOverlayWindowViews::OnUpdateControlsBounds() {
         global_media_controls::DragState::kDragStarted;
     play_pause_controls_view_->SetVisible(show_play_pause_button_ &&
                                           !is_dragging_progress_bar);
-    replay_10_seconds_button_->SetVisible(!is_dragging_progress_bar);
-    forward_10_seconds_button_->SetVisible(!is_dragging_progress_bar);
+    replay_10_seconds_button_->SetVisible(false);
+    forward_10_seconds_button_->SetVisible(false);
 
     // The previous and next track buttons are always visible, but disabled if
     // there is no action handler for them.
     previous_track_controls_view_->SetEnabled(show_previous_track_button_);
     next_track_controls_view_->SetEnabled(show_next_track_button_);
+    previous_track_controls_view_->SetVisible(false);
+    next_track_controls_view_->SetVisible(false);
+    timestamp_->SetVisible(false);
     return;
   }
 
