diff --git a/chrome/installer/util/logging_installer.cc b/chrome/installer/util/logging_installer.cc
--- a/chrome/installer/util/logging_installer.cc
+++ b/chrome/installer/util/logging_installer.cc
@@ -126,7 +126,7 @@
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
       FILE_PATH_LITERAL("chrome_installer.log");
 #else  // BUILDFLAG(CHROMIUM_BRANDING)
-      FILE_PATH_LITERAL("chromium_installer.log");
+      FILE_PATH_LITERAL("dao_installer.log");
 #endif
 
   // Fallback to current directory if getting the secure or temp directory
