diff --git a/chrome/common/url_constants.h b/chrome/common/url_constants.h
index 012f27fbd5..ea7fae12c7 100644
--- a/chrome/common/url_constants.h
+++ b/chrome/common/url_constants.h
@@ -476,11 +476,11 @@ inline constexpr char kPrivacyLearnMoreURL[] =
 
 // "Chrome Settings" URL for Ad Topics page
 inline constexpr char kPrivacySandboxAdTopicsURL[] =
-    "chrome://settings/adPrivacy/interests";
+    "dao://settings/adPrivacy/interests";
 
 // "Chrome Settings" URL for Managing Topics page
 inline constexpr char kPrivacySandboxManageTopicsURL[] =
-    "chrome://settings/adPrivacy/interests/manage";
+    "dao://settings/adPrivacy/interests/manage";
 
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
 // The Privacy Sandbox homepage.
