diff --git a/chrome/common/webui_url_constants.h b/chrome/common/webui_url_constants.h
index e128efef6cf36..6929e304ccdec 100644
--- a/chrome/common/webui_url_constants.h
+++ b/chrome/common/webui_url_constants.h
@@ -444,7 +444,7 @@ inline constexpr char kChromeUIWebAppInternalsHost[] = "web-app-internals";
 inline constexpr char kChromeUIWebuiBrowserHost[] = "webui-browser";
 inline constexpr char kChromeUIWebuiBrowserURL[] = "chrome://webui-browser/";
 inline constexpr char kChromeUIWebUIJsErrorHost[] = "webuijserror";
-inline constexpr char kChromeUIWebUIJsErrorURL[] = "chrome://webuijserror/";
+inline constexpr char kChromeUIWebUIJsErrorURL[] = "dao://webuijserror/";
 inline constexpr char kChromeUIYourSavedInfoPath[] = "/yourSavedInfo";
 inline constexpr char kCookiesSubPagePath[] = "/cookies";
 #endif  // BUILDFLAG(IS_ANDROID)
@@ -484,29 +484,29 @@ inline constexpr char kChromeUISandboxHost[] = "sandbox";
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
 inline constexpr char kChromeUIBrowserSwitchHost[] = "browser-switch";
-inline constexpr char kChromeUIBrowserSwitchURL[] = "chrome://browser-switch/";
+inline constexpr char kChromeUIBrowserSwitchURL[] = "dao://browser-switch/";
 inline constexpr char kChromeUIIntroDefaultBrowserSubPage[] = "default-browser";
 inline constexpr char kChromeUIIntroDefaultBrowserURL[] =
-    "chrome://intro/default-browser";
+    "dao://intro/default-browser";
 inline constexpr char kChromeUIIntroSignInCelebrationSubPage[] =
     "sign-in-celebration";
 inline constexpr char kChromeUIIntroHost[] = "intro";
-inline constexpr char kChromeUIIntroURL[] = "chrome://intro";
+inline constexpr char kChromeUIIntroURL[] = "dao://intro";
 inline constexpr char kChromeUIManagedUserProfileNoticeHost[] =
     "managed-user-profile-notice";
 inline constexpr char kChromeUIManagedUserProfileNoticeUrl[] =
-    "chrome://managed-user-profile-notice/";
+    "dao://managed-user-profile-notice/";
 inline constexpr char kChromeUIManagedUserProfileNoticeRefreshURL[] =
-    "chrome://managed-user-profile-notice/"
+    "dao://managed-user-profile-notice/"
     "managed_user_profile_notice_refresh.html";
 inline constexpr char kChromeUIProfileCustomizationHost[] =
     "profile-customization";
 inline constexpr char kChromeUIProfileCustomizationURL[] =
-    "chrome://profile-customization";
+    "dao://profile-customization";
 inline constexpr char kChromeUIProfilePickerHost[] = "profile-picker";
 inline constexpr char kChromeUIProfilePickerStartupQuery[] = "startup";
 inline constexpr char kChromeUIProfilePickerGlicQuery[] = "glic";
-inline constexpr char kChromeUIProfilePickerUrl[] = "chrome://profile-picker/";
+inline constexpr char kChromeUIProfilePickerUrl[] = "dao://profile-picker/";
 inline constexpr char kChromeUIHistorySyncOptinHost[] = "history-sync-optin";
 inline constexpr char kChromeUIHistorySyncOptinURL[] =
     "chrome://history-sync-optin/";
@@ -619,11 +619,11 @@ inline constexpr char kChromeUICastFeedbackURL[] = "chrome://cast-feedback";
 
 #if BUILDFLAG(CHROME_ROOT_STORE_CERT_MANAGEMENT_UI)
 inline constexpr char kChromeUICertificateManagerDialogURL[] =
-    "chrome://certificate-manager";
+    "dao://certificate-manager";
 inline constexpr char kChromeUICertificateManagerHost[] = "certificate-manager";
 inline constexpr char kChromeUICertificateRedirectPath[] = "/certificates";
 inline constexpr char kChromeUICertificateRedirectURL[] =
-    "chrome://settings/certificates";
+    "dao://settings/certificates";
 #endif  // BUILDFLAG(CHROME_ROOT_STORE_CERT_MANAGEMENT_UI)
 
 #if BUILDFLAG(IS_MAC)
