diff --git a/chrome/common/webui_url_constants.h b/chrome/common/webui_url_constants.h
index ff691df3c5..181ced822b 100644
--- a/chrome/common/webui_url_constants.h
+++ b/chrome/common/webui_url_constants.h
@@ -26,61 +26,61 @@
 namespace chrome {
 
 // chrome: components (without schemes) and URLs (including schemes).
-// e.g. kChromeUIFooHost = "foo" and kChromeUIFooURL = "chrome://foo/"
+// e.g. kChromeUIFooHost = "foo" and kChromeUIFooURL = "dao://foo/"
 // Not all components have corresponding URLs and vice versa. Only add as
 // needed.
 // Please keep in alphabetical order, with OS/feature specific sections below.
 inline constexpr char kChromeUIAboutHost[] = "about";
-inline constexpr char kChromeUIAboutURL[] = "chrome://about/";
+inline constexpr char kChromeUIAboutURL[] = "dao://about/";
 inline constexpr char kChromeUIAccessCodeCastHost[] = "access-code-cast";
 inline constexpr char kChromeUIAccessCodeCastURL[] =
-    "chrome://access-code-cast/";
+    "dao://access-code-cast/";
 inline constexpr char kChromeUIAccessibilityHost[] = "accessibility";
 inline constexpr char kChromeUIActivateSafetyCheckSettingsURL[] =
-    "chrome://settings/safetyCheck?activateSafetyCheck";
+    "dao://settings/safetyCheck?activateSafetyCheck";
 inline constexpr char kChromeUIAllSitesPath[] = "/content/all";
 inline constexpr char kChromeUIAppIconHost[] = "app-icon";
-inline constexpr char kChromeUIAppIconURL[] = "chrome://app-icon/";
+inline constexpr char kChromeUIAppIconURL[] = "dao://app-icon/";
 inline constexpr char kChromeUIAppLauncherPageHost[] = "apps";
-inline constexpr char kChromeUIAppsURL[] = "chrome://apps/";
+inline constexpr char kChromeUIAppsURL[] = "dao://apps/";
 inline constexpr char kChromeUIAppsWithDeprecationDialogURL[] =
-    "chrome://apps?showDeletionDialog=";
+    "dao://apps?showDeletionDialog=";
 inline constexpr char kChromeUIAppsWithForceInstalledDeprecationDialogURL[] =
-    "chrome://apps?showForceInstallDialog=";
+    "dao://apps?showForceInstallDialog=";
 inline constexpr char kChromeUIAutofillInternalsHost[] = "autofill-internals";
 #if BUILDFLAG(ENABLE_DICE_SUPPORT)
 inline constexpr char kChromeUIBatchUploadHost[] = "batch-upload";
-inline constexpr char kChromeUIBatchUploadURL[] = "chrome://batch-upload/";
+inline constexpr char kChromeUIBatchUploadURL[] = "dao://batch-upload/";
 #endif
 inline constexpr char kChromeUIBluetoothInternalsHost[] = "bluetooth-internals";
 inline constexpr char kChromeUIBookmarksHost[] = "bookmarks";
-inline constexpr char kChromeUIBookmarksURL[] = "chrome://bookmarks/";
+inline constexpr char kChromeUIBookmarksURL[] = "dao://bookmarks/";
 inline constexpr char kChromeUIBrowsingTopicsInternalsHost[] =
     "topics-internals";
 inline constexpr char kChromeUICertificateViewerHost[] = "view-cert";
-inline constexpr char kChromeUICertificateViewerURL[] = "chrome://view-cert/";
+inline constexpr char kChromeUICertificateViewerURL[] = "dao://view-cert/";
 inline constexpr char kChromeUIChromeSigninHost[] = "chrome-signin";
-inline constexpr char kChromeUIChromeSigninURL[] = "chrome://chrome-signin/";
+inline constexpr char kChromeUIChromeSigninURL[] = "dao://chrome-signin/";
 inline constexpr char kChromeUIChromeURLsHost[] = "chrome-urls";
-inline constexpr char kChromeUIChromeURLsURL[] = "chrome://chrome-urls/";
-inline constexpr char16_t kChromeUIChromeURLsURL16[] = u"chrome://chrome-urls/";
+inline constexpr char kChromeUIChromeURLsURL[] = "dao://chrome-urls/";
+inline constexpr char16_t kChromeUIChromeURLsURL16[] = u"dao://chrome-urls/";
 inline constexpr char kChromeUIComponentsHost[] = "components";
-inline constexpr char kChromeUIComponentsUrl[] = "chrome://components";
+inline constexpr char kChromeUIComponentsUrl[] = "dao://components";
 inline constexpr char kChromeUIConflictsHost[] = "conflicts";
 inline constexpr char kChromeUIConstrainedHTMLTestURL[] =
-    "chrome://constrained-test/";
+    "dao://constrained-test/";
 inline constexpr char kChromeUIContentSettingsURL[] =
-    "chrome://settings/content";
+    "dao://settings/content";
 inline constexpr char16_t kChromeUIContentSettingsURL16[] =
-    u"chrome://settings/content";
+    u"dao://settings/content";
 inline constexpr char16_t kChromeUICookieSettingsURL[] =
-    u"chrome://settings/cookies";
+    u"dao://settings/cookies";
 inline constexpr char kChromeUICrashesHost[] = "crashes";
-inline constexpr char kChromeUICrashesUrl[] = "chrome://crashes";
+inline constexpr char kChromeUICrashesUrl[] = "dao://crashes";
 inline constexpr char kChromeUICrashHost[] = "crash";
 inline constexpr char kChromeUICreditsHost[] = "credits";
-inline constexpr char kChromeUICreditsURL[] = "chrome://credits/";
-inline constexpr char16_t kChromeUICreditsURL16[] = u"chrome://credits/";
+inline constexpr char kChromeUICreditsURL[] = "dao://credits/";
+inline constexpr char16_t kChromeUICreditsURL16[] = u"dao://credits/";
 inline constexpr char kChromeUIDataSharingInternalsHost[] =
     "data-sharing-internals";
 inline constexpr char kChromeUIDefaultHost[] = "version";
@@ -96,60 +96,60 @@ inline constexpr char kChromeUIDevToolsURL[] =
 inline constexpr char kChromeUIDiceWebSigninInterceptChromeSigninSubPage[] =
     "chrome-signin";
 inline constexpr char kChromeUIDiceWebSigninInterceptChromeSigninURL[] =
-    "chrome://signin-dice-web-intercept.top-chrome/chrome-signin";
+    "dao://signin-dice-web-intercept.top-chrome/chrome-signin";
 inline constexpr char kChromeUIDiceWebSigninInterceptHost[] =
     "signin-dice-web-intercept.top-chrome";
 inline constexpr char kChromeUIDiceWebSigninInterceptURL[] =
-    "chrome://signin-dice-web-intercept.top-chrome/";
+    "dao://signin-dice-web-intercept.top-chrome/";
 inline constexpr char kChromeUIDownloadInternalsHost[] = "download-internals";
 inline constexpr char kChromeUIDownloadsHost[] = "downloads";
-inline constexpr char kChromeUIDownloadsURL[] = "chrome://downloads/";
+inline constexpr char kChromeUIDownloadsURL[] = "dao://downloads/";
 inline constexpr char kChromeUIDriveInternalsHost[] = "drive-internals";
 inline constexpr char kChromeUIEDUCoexistenceLoginURLV2[] =
-    "chrome://chrome-signin/edu-coexistence";
+    "dao://chrome-signin/edu-coexistence";
 inline constexpr char kChromeUIExtensionIconHost[] = "extension-icon";
-inline constexpr char kChromeUIExtensionIconURL[] = "chrome://extension-icon/";
+inline constexpr char kChromeUIExtensionIconURL[] = "dao://extension-icon/";
 inline constexpr char kChromeUIExtensionsHost[] = "extensions";
 inline constexpr char kChromeUIExtensionsInternalsHost[] =
     "extensions-internals";
-inline constexpr char kChromeUIExtensionsURL[] = "chrome://extensions/";
+inline constexpr char kChromeUIExtensionsURL[] = "dao://extensions/";
 inline constexpr char kChromeUIFamilyLinkUserInternalsHost[] =
     "family-link-user-internals";
 inline constexpr char kChromeUIFavicon2Host[] = "favicon2";
 inline constexpr char kChromeUIFaviconHost[] = "favicon";
-inline constexpr char kChromeUIFaviconURL[] = "chrome://favicon/";
+inline constexpr char kChromeUIFaviconURL[] = "dao://favicon/";
 inline constexpr char kChromeUIFeedbackHost[] = "feedback";
-inline constexpr char kChromeUIFeedbackURL[] = "chrome://feedback/";
-inline constexpr char kChromeUIFileiconURL[] = "chrome://fileicon/";
+inline constexpr char kChromeUIFeedbackURL[] = "dao://feedback/";
+inline constexpr char kChromeUIFileiconURL[] = "dao://fileicon/";
 inline constexpr char kChromeUIFlagsHost[] = "flags";
-inline constexpr char kChromeUIFlagsURL[] = "chrome://flags/";
-inline constexpr char16_t kChromeUIFlagsURL16[] = u"chrome://flags/";
+inline constexpr char kChromeUIFlagsURL[] = "dao://flags/";
+inline constexpr char16_t kChromeUIFlagsURL16[] = u"dao://flags/";
 inline constexpr char kChromeUIGCMInternalsHost[] = "gcm-internals";
 inline constexpr char kChromeUIGlicHost[] = "glic";
-inline constexpr char kChromeUIGlicURL[] = "chrome://glic/";
-inline constexpr char kChromeUIGlicFreURL[] = "chrome://glic/glic_fre/fre.html";
+inline constexpr char kChromeUIGlicURL[] = "dao://glic/";
+inline constexpr char kChromeUIGlicFreURL[] = "dao://glic/glic_fre/fre.html";
 inline constexpr char kChromeUIHangUIHost[] = "uithreadhang";
 inline constexpr char kChromeUIHelpHost[] = "help";
-inline constexpr char kChromeUIHelpURL[] = "chrome://help/";
+inline constexpr char kChromeUIHelpURL[] = "dao://help/";
 inline constexpr char kChromeUIHistoryHost[] = "history";
 inline constexpr char kChromeUIHistorySyncedTabs[] = "/syncedTabs";
-inline constexpr char kChromeUIHistoryURL[] = "chrome://history/";
-inline constexpr char16_t kChromeUIHistoryURL16[] = u"chrome://history/";
+inline constexpr char kChromeUIHistoryURL[] = "dao://history/";
+inline constexpr char16_t kChromeUIHistoryURL16[] = u"dao://history/";
 inline constexpr char kChromeUIIdentityInternalsHost[] = "identity-internals";
 inline constexpr char kChromeUIImageHost[] = "image";
-inline constexpr char kChromeUIImageURL[] = "chrome://image/";
+inline constexpr char kChromeUIImageURL[] = "dao://image/";
 inline constexpr char kChromeUIInspectHost[] = "inspect";
-inline constexpr char kChromeUIInspectURL[] = "chrome://inspect/";
+inline constexpr char kChromeUIInspectURL[] = "dao://inspect/";
 inline constexpr char kChromeUIInternalsHost[] = "internals";
 inline constexpr char kChromeUIInterstitialHost[] = "interstitials";
-inline constexpr char kChromeUIInterstitialURL[] = "chrome://interstitials/";
+inline constexpr char kChromeUIInterstitialURL[] = "dao://interstitials/";
 inline constexpr char kChromeUIKillHost[] = "kill";
 inline constexpr char kChromeUILauncherInternalsHost[] = "launcher-internals";
 inline constexpr char kChromeUILocalStateHost[] = "local-state";
 inline constexpr char kChromeUILocationInternalsHost[] = "location-internals";
 inline constexpr char kChromeUIManagementHost[] = "management";
-inline constexpr char kChromeUIManagementURL[] = "chrome://management";
-inline constexpr char16_t kChromeUIManagementURL16[] = u"chrome://management";
+inline constexpr char kChromeUIManagementURL[] = "dao://management";
+inline constexpr char16_t kChromeUIManagementURL16[] = u"dao://management";
 inline constexpr char kChromeUIMediaEngagementHost[] = "media-engagement";
 inline constexpr char kChromeUIMediaRouterInternalsHost[] =
     "media-router-internals";
@@ -158,15 +158,15 @@ inline constexpr char kChromeUIMetricsInternalsHost[] = "metrics-internals";
 inline constexpr char kChromeUINaClHost[] = "nacl";
 inline constexpr char kChromeUINetExportHost[] = "net-export";
 inline constexpr char kChromeUINetInternalsHost[] = "net-internals";
-inline constexpr char kChromeUINetInternalsURL[] = "chrome://net-internals/";
+inline constexpr char kChromeUINetInternalsURL[] = "dao://net-internals/";
 inline constexpr char kChromeUINewTabHost[] = "newtab";
 inline constexpr char kChromeUINewTabPageHost[] = "new-tab-page";
 inline constexpr char kChromeUINewTabPageThirdPartyHost[] =
     "new-tab-page-third-party";
 inline constexpr char kChromeUINewTabPageThirdPartyURL[] =
-    "chrome://new-tab-page-third-party/";
-inline constexpr char kChromeUINewTabPageURL[] = "chrome://new-tab-page/";
-inline constexpr char kChromeUINewTabURL[] = "chrome://newtab/";
+    "dao://new-tab-page-third-party/";
+inline constexpr char kChromeUINewTabPageURL[] = "dao://new-tab-page/";
+inline constexpr char kChromeUINewTabURL[] = "dao://newtab/";
 inline constexpr char kChromeUIUntrustedNtpMicrosoftAuthHost[] =
     "ntp-microsoft-auth";
 inline constexpr char kChromeUIUntrustedNtpMicrosoftAuthURL[] =
@@ -175,26 +175,26 @@ inline constexpr char kChromeUINTPTilesInternalsHost[] = "ntp-tiles-internals";
 inline constexpr char kChromeUIOmniboxHost[] = "omnibox";
 inline constexpr char kChromeUIOmniboxPopupHost[] = "omnibox-popup.top-chrome";
 inline constexpr char kChromeUIOmniboxPopupURL[] =
-    "chrome://omnibox-popup.top-chrome/";
-inline constexpr char kChromeUIOmniboxURL[] = "chrome://omnibox/";
+    "dao://omnibox-popup.top-chrome/";
+inline constexpr char kChromeUIOmniboxURL[] = "dao://omnibox/";
 inline constexpr char kChromeUIOnDeviceTranslationInternalsHost[] =
     "on-device-translation-internals";
 inline constexpr char kChromeUIPasswordManagerCheckupURL[] =
-    "chrome://password-manager/checkup?start=true";
+    "dao://password-manager/checkup?start=true";
 inline constexpr char kChromeUIPasswordManagerInternalsHost[] =
     "password-manager-internals";
 inline constexpr char kChromeUIPasswordManagerSettingsURL[] =
-    "chrome://password-manager/settings";
+    "dao://password-manager/settings";
 inline constexpr char kChromeUIPasswordManagerURL[] =
-    "chrome://password-manager";
+    "dao://password-manager";
 inline constexpr char kChromeUiPasswordChangeUrl[] =
-    "chrome://password-manager/settings/password-change";
+    "dao://password-manager/settings/password-change";
 inline constexpr char kChromeUIPolicyHost[] = "policy";
-inline constexpr char kChromeUIPolicyTestURL[] = "chrome://policy/test";
-inline constexpr char kChromeUIPolicyURL[] = "chrome://policy/";
+inline constexpr char kChromeUIPolicyTestURL[] = "dao://policy/test";
+inline constexpr char kChromeUIPolicyURL[] = "dao://policy/";
 inline constexpr char kChromeUIPredictorsHost[] = "predictors";
 inline constexpr char kChromeUIPrefsInternalsHost[] = "prefs-internals";
-inline constexpr char kChromeUIPrintURL[] = "chrome://print/";
+inline constexpr char kChromeUIPrintURL[] = "dao://print/";
 inline constexpr char kChromeUIPrivacySandboxDialogCombinedPath[] = "combined";
 inline constexpr char kChromeUIPrivacySandboxDialogHost[] =
     "privacy-sandbox-dialog";
@@ -202,24 +202,24 @@ inline constexpr char kChromeUIPrivacySandboxDialogNoticePath[] = "notice";
 inline constexpr char kChromeUIPrivacySandboxDialogNoticeRestrictedPath[] =
     "restricted";
 inline constexpr char kChromeUIPrivacySandboxDialogURL[] =
-    "chrome://privacy-sandbox-dialog";
+    "dao://privacy-sandbox-dialog";
 inline constexpr char16_t kChromeUIPrivacySandboxFledgeURL[] =
-    u"chrome://settings/adPrivacy/sites";
+    u"dao://settings/adPrivacy/sites";
 inline constexpr char kChromeUIPrivacySandboxInternalsHost[] =
     "privacy-sandbox-internals";
 inline constexpr char kChromeUIPrivacySandboxInternalsURL[] =
-    "chrome://privacy-sandbox-internals";
+    "dao://privacy-sandbox-internals";
 inline constexpr char16_t kChromeUIPrivacySandboxManageTopicsLearnMoreURL[] =
     u"https://support.google.com/chrome?p=ad_privacy";
 inline constexpr char16_t kChromeUIPrivacySandboxTopicsURL[] =
-    u"chrome://settings/adPrivacy/interests";
+    u"dao://settings/adPrivacy/interests";
 inline constexpr char kChromeUIProfileInternalsHost[] = "profile-internals";
 inline constexpr char kChromeUIQuitHost[] = "quit";
-inline constexpr char kChromeUIQuitURL[] = "chrome://quit/";
+inline constexpr char kChromeUIQuitURL[] = "dao://quit/";
 inline constexpr char kChromeUIResetPasswordHost[] = "reset-password";
-inline constexpr char kChromeUIResetPasswordURL[] = "chrome://reset-password/";
+inline constexpr char kChromeUIResetPasswordURL[] = "dao://reset-password/";
 inline constexpr char kChromeUIRestartHost[] = "restart";
-inline constexpr char kChromeUIRestartURL[] = "chrome://restart/";
+inline constexpr char kChromeUIRestartURL[] = "dao://restart/";
 inline constexpr char kChromeUISafetyPixelbookURL[] =
     "https://g.co/Pixelbook/legal";
 inline constexpr char kChromeUISafetyPixelSlateURL[] =
@@ -231,38 +231,38 @@ inline constexpr char kChromeUISegmentationInternalsHost[] =
 inline constexpr char kChromeUISensorInfoHost[] = "sensor-info";
 inline constexpr char kChromeUISettingsHost[] = "settings";
 inline constexpr char16_t kChromeUISettingsHost16[] = u"settings";
-inline constexpr char kChromeUISettingsURL[] = "chrome://settings/";
-inline constexpr char16_t kChromeUISettingsURL16[] = u"chrome://settings/";
+inline constexpr char kChromeUISettingsURL[] = "dao://settings/";
+inline constexpr char16_t kChromeUISettingsURL16[] = u"dao://settings/";
 inline constexpr char kChromeUISigninEmailConfirmationHost[] =
     "signin-email-confirmation";
 inline constexpr char kChromeUISigninEmailConfirmationURL[] =
-    "chrome://signin-email-confirmation";
+    "dao://signin-email-confirmation";
 inline constexpr char kChromeUISigninErrorHost[] = "signin-error";
-inline constexpr char kChromeUISigninErrorURL[] = "chrome://signin-error/";
+inline constexpr char kChromeUISigninErrorURL[] = "dao://signin-error/";
 inline constexpr char kChromeUISignInInternalsHost[] = "signin-internals";
 #if BUILDFLAG(ENABLE_DICE_SUPPORT)
 inline constexpr char kChromeUISignoutConfirmationHost[] =
     "signout-confirmation";
 inline constexpr char kChromeUISignoutConfirmationURL[] =
-    "chrome://signout-confirmation";
+    "dao://signout-confirmation";
 #endif
 inline constexpr char kChromeUISiteEngagementHost[] = "site-engagement";
 inline constexpr char kChromeUISuggestInternalsHost[] = "suggest-internals";
 inline constexpr char kChromeUISuggestInternalsURL[] =
-    "chrome://suggest-internals/";
+    "dao://suggest-internals/";
 inline constexpr char kChromeUISupervisedUserPassphrasePageHost[] =
     "managed-user-passphrase";
 inline constexpr char kChromeUISupportToolHost[] = "support-tool";
 inline constexpr char kChromeUISyncConfirmationHost[] = "sync-confirmation";
 inline constexpr char kChromeUISyncConfirmationLoadingPath[] = "loading";
 inline constexpr char kChromeUISyncConfirmationURL[] =
-    "chrome://sync-confirmation/";
+    "dao://sync-confirmation/";
 inline constexpr char kChromeUISyncInternalsHost[] = "sync-internals";
 inline constexpr char kChromeUISystemInfoHost[] = "system";
 inline constexpr char kChromeUITermsHost[] = "terms";
-inline constexpr char kChromeUITermsURL[] = "chrome://terms/";
+inline constexpr char kChromeUITermsURL[] = "dao://terms/";
 inline constexpr char kChromeUIThemeHost[] = "theme";
-inline constexpr char kChromeUIThemeURL[] = "chrome://theme/";
+inline constexpr char kChromeUIThemeURL[] = "dao://theme/";
 inline constexpr char kChromeUITopChromeDomain[] = "top-chrome";
 inline constexpr char kChromeUITranslateInternalsHost[] = "translate-internals";
 inline constexpr char kChromeUIUntrustedComposeHost[] = "compose";
@@ -291,15 +291,15 @@ inline constexpr char kChromeUIUserActionsHost[] = "user-actions";
 inline constexpr char kChromeUIUserEducationInternalsHost[] =
     "user-education-internals";
 inline constexpr char kChromeUIUserEducationInternalsURL[] =
-    "chrome://user-education-internals";
+    "dao://user-education-internals";
 inline constexpr char kChromeUIVersionHost[] = "version";
-inline constexpr char kChromeUIVersionURL[] = "chrome://version/";
-inline constexpr char16_t kChromeUIVersionURL16[] = u"chrome://version/";
+inline constexpr char kChromeUIVersionURL[] = "dao://version/";
+inline constexpr char16_t kChromeUIVersionURL16[] = u"dao://version/";
 inline constexpr char kChromeUIWebRtcLogsHost[] = "webrtc-logs";
 inline constexpr char kChromeUIWebuiGalleryHost[] = "webui-gallery";
 
 #if BUILDFLAG(IS_ANDROID)
-inline constexpr char kChromeUIJavaCrashURL[] = "chrome://java-crash/";
+inline constexpr char kChromeUIJavaCrashURL[] = "dao://java-crash/";
 inline constexpr char kChromeUINativeBookmarksURL[] =
     "chrome-native://bookmarks/";
 inline constexpr char kChromeUINativeExploreURL[] = "chrome-native://explore";
@@ -313,15 +313,15 @@ inline constexpr char kChromeUIAppServiceInternalsHost[] =
 inline constexpr char kChromeUIBookmarksSidePanelHost[] =
     "bookmarks-side-panel.top-chrome";
 inline constexpr char kChromeUIBookmarksSidePanelURL[] =
-    "chrome://bookmarks-side-panel.top-chrome/";
+    "dao://bookmarks-side-panel.top-chrome/";
 inline constexpr char kChromeUICustomizeChromeSidePanelHost[] =
     "customize-chrome-side-panel.top-chrome";
 inline constexpr char kChromeUICustomizeChromeSidePanelURL[] =
-    "chrome://customize-chrome-side-panel.top-chrome";
+    "dao://customize-chrome-side-panel.top-chrome";
 inline constexpr char kChromeUIHistoryClustersSidePanelHost[] =
     "history-clusters-side-panel.top-chrome";
 inline constexpr char kChromeUIHistoryClustersSidePanelURL[] =
-    "chrome://history-clusters-side-panel.top-chrome/";
+    "dao://history-clusters-side-panel.top-chrome/";
 inline constexpr char kChromeUILensHost[] = "lens";
 inline constexpr char kChromeUILensSidePanelHost[] = "lens";
 inline constexpr char kChromeUILensUntrustedSidePanelAPIURL[] =
@@ -333,18 +333,18 @@ inline constexpr char kChromeUILensOverlayUntrustedURL[] =
     "chrome-untrusted://lens-overlay/";
 inline constexpr char kChromeUINearbyInternalsHost[] = "nearby-internals";
 inline constexpr char kChromeUINearbyShareHost[] = "nearby";
-inline constexpr char kChromeUINearbyShareURL[] = "chrome://nearby/";
+inline constexpr char kChromeUINearbyShareURL[] = "dao://nearby/";
 inline constexpr char kChromeUIOnDeviceInternalsHost[] = "on-device-internals";
 inline constexpr char kChromeUIReadLaterHost[] = "read-later.top-chrome";
 inline constexpr char kChromeUIReadLaterURL[] =
-    "chrome://read-later.top-chrome/";
+    "dao://read-later.top-chrome/";
 inline constexpr char kChromeUISearchEngineChoiceHost[] =
     "search-engine-choice";
 inline constexpr char kChromeUISearchEngineChoiceURL[] =
-    "chrome://search-engine-choice";
+    "dao://search-engine-choice";
 inline constexpr char kChromeUITabSearchHost[] = "tab-search.top-chrome";
 inline constexpr char kChromeUITabSearchURL[] =
-    "chrome://tab-search.top-chrome/";
+    "dao://tab-search.top-chrome/";
 inline constexpr char kChromeUIUntrustedFeedURL[] = "chrome-untrusted://feed/";
 inline constexpr char kChromeUIUntrustedReadAnythingSidePanelHost[] =
     "read-anything-side-panel.top-chrome";
@@ -354,7 +354,7 @@ inline constexpr char kChromeUIUntrustedWebUITestURL[] =
     "chrome-untrusted://webui-test/";
 inline constexpr char kChromeUIWebAppInternalsHost[] = "web-app-internals";
 inline constexpr char kChromeUIWebUIJsErrorHost[] = "webuijserror";
-inline constexpr char kChromeUIWebUIJsErrorURL[] = "chrome://webuijserror/";
+inline constexpr char kChromeUIWebUIJsErrorURL[] = "dao://webuijserror/";
 inline constexpr char kChromeUIWebUITestHost[] = "webui-test";
 inline constexpr char kCookiesSubPagePath[] = "/cookies";
 inline constexpr char kTrackingProtectionSubPagePath[] = "/trackingProtection";
@@ -366,114 +366,114 @@ inline constexpr char kTrackingProtectionSubPagePath[] = "/trackingProtection";
 inline constexpr char kChromeUIAccountManagerErrorHost[] =
     "account-manager-error";
 inline constexpr char kChromeUIAccountManagerErrorURL[] =
-    "chrome://account-manager-error";
+    "dao://account-manager-error";
 inline constexpr char kChromeUIAccountMigrationWelcomeHost[] =
     "account-migration-welcome";
 inline constexpr char kChromeUIAccountMigrationWelcomeURL[] =
-    "chrome://account-migration-welcome";
+    "dao://account-migration-welcome";
 inline constexpr char kChromeUIAddSupervisionHost[] = "add-supervision";
 inline constexpr char kChromeUIAddSupervisionURL[] =
-    "chrome://add-supervision/";
+    "dao://add-supervision/";
 inline constexpr char kChromeUIAppDisabledHost[] = "app-disabled";
 inline constexpr char kChromeUIAppInstallDialogHost[] = "app-install-dialog";
 inline constexpr char kChromeUIAppInstallDialogURL[] =
-    "chrome://app-install-dialog/";
+    "dao://app-install-dialog/";
 inline constexpr char kChromeUIArcOverviewTracingHost[] =
     "arc-overview-tracing";
 inline constexpr char kChromeUIArcPowerControlHost[] = "arc-power-control";
 inline constexpr char kChromeUIAssistantOptInHost[] = "assistant-optin";
 inline constexpr char kChromeUIAssistantOptInURL[] =
-    "chrome://assistant-optin/";
+    "dao://assistant-optin/";
 inline constexpr char kChromeUIAudioHost[] = "audio";
-inline constexpr char kChromeUIAudioURL[] = "chrome://audio/";
+inline constexpr char kChromeUIAudioURL[] = "dao://audio/";
 inline constexpr char kChromeUIBluetoothPairingHost[] = "bluetooth-pairing";
 inline constexpr char kChromeUIBluetoothPairingURL[] =
-    "chrome://bluetooth-pairing/";
+    "dao://bluetooth-pairing/";
 inline constexpr char kChromeUIBorealisCreditsHost[] = "borealis-credits";
 inline constexpr char kChromeUIBorealisInstallerHost[] = "borealis-installer";
 inline constexpr char kChromeUIBorealisInstallerUrl[] =
-    "chrome://borealis-installer";
+    "dao://borealis-installer";
 inline constexpr char kChromeUICloudUploadHost[] = "cloud-upload";
-inline constexpr char kChromeUICloudUploadURL[] = "chrome://cloud-upload/";
+inline constexpr char kChromeUICloudUploadURL[] = "dao://cloud-upload/";
 inline constexpr char kChromeUIConfirmPasswordChangeHost[] =
     "confirm-password-change";
 inline constexpr char kChromeUIConfirmPasswordChangeUrl[] =
-    "chrome://confirm-password-change";
+    "dao://confirm-password-change";
 inline constexpr char kChromeUICrostiniCreditsHost[] = "crostini-credits";
 inline constexpr char16_t kChromeUICrostiniCreditsURL16[] =
-    u"chrome://crostini-credits/";
+    u"dao://crostini-credits/";
 inline constexpr char kChromeUICrostiniInstallerHost[] = "crostini-installer";
 inline constexpr char kChromeUICrostiniInstallerUrl[] =
-    "chrome://crostini-installer";
+    "dao://crostini-installer";
 inline constexpr char kChromeUICrostiniUpgraderHost[] = "crostini-upgrader";
 inline constexpr char kChromeUICrostiniUpgraderUrl[] =
-    "chrome://crostini-upgrader";
+    "dao://crostini-upgrader";
 inline constexpr char kChromeUICryptohomeHost[] = "cryptohome";
 inline constexpr char kChromeUIDeviceEmulatorHost[] = "device-emulator";
 inline constexpr char kChromeUIEmojiPickerHost[] = "emoji-picker";
-inline constexpr char kChromeUIEmojiPickerURL[] = "chrome://emoji-picker/";
+inline constexpr char kChromeUIEmojiPickerURL[] = "dao://emoji-picker/";
 inline constexpr char kChromeUIEnterpriseReportingHost[] =
     "enterprise-reporting";
 inline constexpr char kChromeUIExtendedUpdatesDialogHost[] =
     "extended-updates-dialog";
 inline constexpr char kChromeUIExtendedUpdatesDialogURL[] =
-    "chrome://extended-updates-dialog";
+    "dao://extended-updates-dialog";
 inline constexpr char kChromeUIHealthdInternalsHost[] = "healthd-internals";
 inline constexpr char kChromeUIInternetConfigDialogHost[] =
     "internet-config-dialog";
 inline constexpr char kChromeUIInternetConfigDialogURL[] =
-    "chrome://internet-config-dialog/";
+    "dao://internet-config-dialog/";
 inline constexpr char kChromeUIInternetDetailDialogHost[] =
     "internet-detail-dialog";
 inline constexpr char kChromeUIInternetDetailDialogURL[] =
-    "chrome://internet-detail-dialog/";
+    "dao://internet-detail-dialog/";
 inline constexpr char kChromeUILocalFilesMigrationHost[] =
     "local-files-migration";
 inline constexpr char kChromeUILocalFilesMigrationURL[] =
-    "chrome://local-files-migration/";
+    "dao://local-files-migration/";
 inline constexpr char kChromeUILockScreenNetworkHost[] = "lock-network";
-inline constexpr char kChromeUILockScreenNetworkURL[] = "chrome://lock-network";
+inline constexpr char kChromeUILockScreenNetworkURL[] = "dao://lock-network";
 inline constexpr char kChromeUILockScreenStartReauthHost[] = "lock-reauth";
 inline constexpr char kChromeUILockScreenStartReauthURL[] =
-    "chrome://lock-reauth";
+    "dao://lock-reauth";
 inline constexpr char kChromeUIManageMirrorSyncHost[] = "manage-mirrorsync";
 inline constexpr char kChromeUIManageMirrorSyncURL[] =
-    "chrome://manage-mirrorsync";
+    "dao://manage-mirrorsync";
 inline constexpr char kChromeUIMobileSetupHost[] = "mobilesetup";
-inline constexpr char kChromeUIMobileSetupURL[] = "chrome://mobilesetup/";
+inline constexpr char kChromeUIMobileSetupURL[] = "dao://mobilesetup/";
 inline constexpr char kChromeUIMultiDeviceInternalsHost[] =
     "multidevice-internals";
 inline constexpr char kChromeUIMultiDeviceSetupHost[] = "multidevice-setup";
 inline constexpr char kChromeUIMultiDeviceSetupUrl[] =
-    "chrome://multidevice-setup";
+    "dao://multidevice-setup";
 inline constexpr char kChromeUINetworkHost[] = "network";
 inline constexpr char kChromeUINotificationTesterHost[] = "notification-tester";
 inline constexpr char kChromeUIOfficeFallbackHost[] = "office-fallback";
 inline constexpr char kChromeUIOfficeFallbackURL[] =
-    "chrome://office-fallback/";
+    "dao://office-fallback/";
 inline constexpr char kChromeUIOobeHost[] = "oobe";
-inline constexpr char kChromeUIOobeURL[] = "chrome://oobe/";
+inline constexpr char kChromeUIOobeURL[] = "dao://oobe/";
 inline constexpr char kChromeUIOSCreditsHost[] = "os-credits";
-inline constexpr char kChromeUIOSCreditsURL[] = "chrome://os-credits/";
-inline constexpr char16_t kChromeUIOSCreditsURL16[] = u"chrome://os-credits/";
+inline constexpr char kChromeUIOSCreditsURL[] = "dao://os-credits/";
+inline constexpr char16_t kChromeUIOSCreditsURL16[] = u"dao://os-credits/";
 inline constexpr char kChromeUIParentAccessHost[] = "parent-access";
-inline constexpr char kChromeUIParentAccessURL[] = "chrome://parent-access/";
+inline constexpr char kChromeUIParentAccessURL[] = "dao://parent-access/";
 inline constexpr char kChromeUIPasswordChangeHost[] = "password-change";
-inline constexpr char kChromeUIPasswordChangeUrl[] = "chrome://password-change";
+inline constexpr char kChromeUIPasswordChangeUrl[] = "dao://password-change";
 inline constexpr char kChromeUIPowerHost[] = "power";
 inline constexpr char kChromeUIRemoteManagementCurtainHost[] =
     "security-curtain";
-inline constexpr char kChromeUISanitizeAppURL[] = "chrome://sanitize";
+inline constexpr char kChromeUISanitizeAppURL[] = "dao://sanitize";
 inline constexpr char kChromeUISetTimeHost[] = "set-time";
-inline constexpr char kChromeUISetTimeURL[] = "chrome://set-time/";
+inline constexpr char kChromeUISetTimeURL[] = "dao://set-time/";
 inline constexpr char kChromeUISlowHost[] = "slow";
 inline constexpr char kChromeUISlowTraceHost[] = "slow_trace";
-inline constexpr char kChromeUISlowURL[] = "chrome://slow/";
+inline constexpr char kChromeUISlowURL[] = "dao://slow/";
 inline constexpr char kChromeUISmbCredentialsHost[] = "smb-credentials-dialog";
 inline constexpr char kChromeUISmbCredentialsURL[] =
-    "chrome://smb-credentials-dialog/";
+    "dao://smb-credentials-dialog/";
 inline constexpr char kChromeUISmbShareHost[] = "smb-share-dialog";
-inline constexpr char kChromeUISmbShareURL[] = "chrome://smb-share-dialog/";
+inline constexpr char kChromeUISmbShareURL[] = "dao://smb-share-dialog/";
 inline constexpr char kChromeUISysInternalsHost[] = "sys-internals";
 inline constexpr char kChromeUIUntrustedCroshHost[] = "crosh";
 inline constexpr char kChromeUIUntrustedCroshURL[] =
@@ -484,7 +484,7 @@ inline constexpr char kChromeUIUntrustedTerminalURL[] =
 inline constexpr char kChromeUIUrgentPasswordExpiryNotificationHost[] =
     "urgent-password-expiry-notification";
 inline constexpr char kChromeUIUrgentPasswordExpiryNotificationUrl[] =
-    "chrome://urgent-password-expiry-notification/";
+    "dao://urgent-password-expiry-notification/";
 inline constexpr char kChromeUIUserImageHost[] = "userimage";
 inline constexpr char kChromeUIVmHost[] = "vm";
 
@@ -493,18 +493,18 @@ inline constexpr char kChromeUIVmHost[] = "vm";
 // systems would be considered part of the OS or window manager.
 bool IsSystemWebUIHost(std::string_view host);
 
-inline constexpr char kChromeUIAppDisabledURL[] = "chrome://app-disabled";
+inline constexpr char kChromeUIAppDisabledURL[] = "dao://app-disabled";
 inline constexpr char kChromeUIDlpInternalsHost[] = "dlp-internals";
-inline constexpr char kChromeUIHistogramsURL[] = "chrome://histograms";
+inline constexpr char kChromeUIHistogramsURL[] = "dao://histograms";
 inline constexpr char kChromeUIKerberosInBrowserHost[] = "kerberos-in-browser";
 inline constexpr char kChromeUIKerberosInBrowserURL[] =
-    "chrome://kerberos-in-browser";
+    "dao://kerberos-in-browser";
 inline constexpr char kChromeUILocationInternalsURL[] =
-    "chrome://location-internals";
-inline constexpr char kChromeUIOsFlagsAppURL[] = "chrome://flags/";
+    "dao://location-internals";
+inline constexpr char kChromeUIOsFlagsAppURL[] = "dao://flags/";
 inline constexpr char kChromeUIOSSettingsHost[] = "os-settings";
-inline constexpr char kChromeUIOSSettingsURL[] = "chrome://os-settings/";
-inline constexpr char kChromeUIOsUrlAppURL[] = "chrome://internal/";
+inline constexpr char kChromeUIOSSettingsURL[] = "dao://os-settings/";
+inline constexpr char kChromeUIOsUrlAppURL[] = "dao://internal/";
 #endif  // BUILDFLAG(IS_CHROMEOS)
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX) || \
@@ -516,14 +516,14 @@ inline constexpr char kChromeUIConnectorsInternalsHost[] =
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX) || \
     BUILDFLAG(IS_CHROMEOS)
 inline constexpr char kChromeUIDiscardsHost[] = "discards";
-inline constexpr char kChromeUIDiscardsURL[] = "chrome://discards/";
+inline constexpr char kChromeUIDiscardsURL[] = "dao://discards/";
 #endif
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
 inline constexpr char kChromeUIWebAppSettingsHost[] = "app-settings";
-inline constexpr char kChromeUIWebAppSettingsURL[] = "chrome://app-settings/";
+inline constexpr char kChromeUIWebAppSettingsURL[] = "dao://app-settings/";
 inline constexpr char kChromeUIWhatsNewHost[] = "whats-new";
-inline constexpr char kChromeUIWhatsNewURL[] = "chrome://whats-new/";
+inline constexpr char kChromeUIWhatsNewURL[] = "dao://whats-new/";
 #endif
 
 #if BUILDFLAG(IS_POSIX) && !BUILDFLAG(IS_MAC) && !BUILDFLAG(IS_ANDROID)
@@ -537,24 +537,24 @@ inline constexpr char kChromeUISandboxHost[] = "sandbox";
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
 inline constexpr char kChromeUIBrowserSwitchHost[] = "browser-switch";
-inline constexpr char kChromeUIBrowserSwitchURL[] = "chrome://browser-switch/";
+inline constexpr char kChromeUIBrowserSwitchURL[] = "dao://browser-switch/";
 inline constexpr char kChromeUIIntroDefaultBrowserSubPage[] = "default-browser";
 inline constexpr char kChromeUIIntroDefaultBrowserURL[] =
-    "chrome://intro/default-browser";
+    "dao://intro/default-browser";
 inline constexpr char kChromeUIIntroHost[] = "intro";
-inline constexpr char kChromeUIIntroURL[] = "chrome://intro";
+inline constexpr char kChromeUIIntroURL[] = "dao://intro";
 inline constexpr char kChromeUIManagedUserProfileNoticeHost[] =
     "managed-user-profile-notice";
 inline constexpr char kChromeUIManagedUserProfileNoticeUrl[] =
-    "chrome://managed-user-profile-notice/";
+    "dao://managed-user-profile-notice/";
 inline constexpr char kChromeUIProfileCustomizationHost[] =
     "profile-customization";
 inline constexpr char kChromeUIProfileCustomizationURL[] =
-    "chrome://profile-customization";
+    "dao://profile-customization";
 inline constexpr char kChromeUIProfilePickerHost[] = "profile-picker";
 inline constexpr char kChromeUIProfilePickerStartupQuery[] = "startup";
 inline constexpr char kChromeUIProfilePickerGlicQuery[] = "glic";
-inline constexpr char kChromeUIProfilePickerUrl[] = "chrome://profile-picker/";
+inline constexpr char kChromeUIProfilePickerUrl[] = "dao://profile-picker/";
 #endif
 
 #if ((BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS)) && \
@@ -575,7 +575,7 @@ inline constexpr char kChromeUISessionServiceInternalsPath[] =
 
 #if BUILDFLAG(ENABLE_WEBUI_TAB_STRIP)
 inline constexpr char kChromeUITabStripHost[] = "tab-strip.top-chrome";
-inline constexpr char kChromeUITabStripURL[] = "chrome://tab-strip.top-chrome";
+inline constexpr char kChromeUITabStripURL[] = "dao://tab-strip.top-chrome";
 #endif
 
 // Settings sub-pages.
@@ -645,11 +645,11 @@ inline constexpr char kChromeUICastFeedbackHost[] = "cast-feedback";
 
 #if BUILDFLAG(CHROME_ROOT_STORE_CERT_MANAGEMENT_UI)
 inline constexpr char kChromeUICertificateManagerDialogURL[] =
-    "chrome://certificate-manager";
+    "dao://certificate-manager";
 inline constexpr char kChromeUICertificateManagerHost[] = "certificate-manager";
 inline constexpr char kChromeUICertificateRedirectPath[] = "/certificates";
 inline constexpr char kChromeUICertificateRedirectURL[] =
-    "chrome://settings/certificates";
+    "dao://settings/certificates";
 #endif  // BUILDFLAG(CHROME_ROOT_STORE_CERT_MANAGEMENT_UI)
 
 // Extensions sub pages.
