diff --git a/content/public/browser/web_contents_view_delegate.h b/content/public/browser/web_contents_view_delegate.h
--- a/content/public/browser/web_contents_view_delegate.h
+++ b/content/public/browser/web_contents_view_delegate.h
@@ -14,6 +14,7 @@
 
 #include "base/functional/callback_forward.h"
 #include "content/common/content_export.h"
+#include "ui/base/dragdrop/mojom/drag_drop_types.mojom-forward.h"
 #include "ui/gfx/native_ui_types.h"
 
 #if defined(__OBJC__)
@@ -22,6 +23,10 @@
 #endif
 #endif
 
+namespace gfx {
+class PointF;
+}
+
 namespace content {
 class NavigationHandle;
 class RenderFrameHost;
@@ -106,7 +111,10 @@
                                 DropCompletionCallback callback);
 
   // Notifies the delegate that the drag operation has ended.
-  virtual void WebContentsDragEnded();
+  virtual void WebContentsDragEnded(
+      const gfx::PointF& screen_point,
+      ui::mojom::DragOperation operation,
+      bool ended_by_mouse_release);
 
 #if BUILDFLAG(IS_ANDROID)
   virtual bool ShouldShowBlurTransitionAnimation(
