diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
index de4c0ef37f..7f6ad86c31 100644
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -557,6 +557,8 @@ if (is_win) {
       deps += [ ":chromium_updater_privileged_helper" ]
     }
 
+    deps += [ "//dao/browser/mcp:dao_mcp_helper_bundle_data" ]
+
     if (enable_stripping) {
       # At link time, preserve the global symbols specified in the .exports
       # file. All other global symbols will be marked as private. The default
