diff --git a/extensions/common/features/simple_feature.cc b/extensions/common/features/simple_feature.cc
index fec1ab8738..3a1d20f3bd 100644
--- a/extensions/common/features/simple_feature.cc
+++ b/extensions/common/features/simple_feature.cc
@@ -31,6 +31,7 @@
 #include "extensions/common/manifest_handlers/background_info.h"
 #include "extensions/common/mojom/context_type.mojom.h"
 #include "extensions/common/switches.h"
+#include "content/public/common/url_constants.h"
 
 using crx_file::id_util::HashedIdInHex;
 using extensions::mojom::ManifestLocation;
