diff --git a/third_party/blink/public/common/chrome_debug_urls.h b/third_party/blink/public/common/chrome_debug_urls.h
index dd20ad2e64..bbeccd3ced 100644
--- a/third_party/blink/public/common/chrome_debug_urls.h
+++ b/third_party/blink/public/common/chrome_debug_urls.h
@@ -21,69 +21,69 @@ namespace blink {
 // too) should rather be implemented by embedders.
 
 // Full about URLs (including schemes).
-inline constexpr char kChromeUIBadCastCrashURL[] = "chrome://badcastcrash/";
+inline constexpr char kChromeUIBadCastCrashURL[] = "dao://badcastcrash/";
 inline constexpr char kChromeUIBrowserCrashURL[] =
-    "chrome://inducebrowsercrashforrealz/";
+    "dao://inducebrowsercrashforrealz/";
 inline constexpr char kChromeUIBrowserDcheckURL[] =
-    "chrome://inducebrowserdcheckforrealz/";
-inline constexpr char kChromeUIBrowserUIHang[] = "chrome://uithreadhang/";
-inline constexpr char kChromeUICheckCrashURL[] = "chrome://checkcrash/";
-inline constexpr char kChromeUICrashURL[] = "chrome://crash/";
+    "dao://inducebrowserdcheckforrealz/";
+inline constexpr char kChromeUIBrowserUIHang[] = "dao://uithreadhang/";
+inline constexpr char kChromeUICheckCrashURL[] = "dao://checkcrash/";
+inline constexpr char kChromeUICrashURL[] = "dao://crash/";
 inline constexpr char kChromeUIDelayedBrowserUIHang[] =
-    "chrome://delayeduithreadhang/";
-inline constexpr char kChromeUIDumpURL[] = "chrome://crashdump/";
-inline constexpr char kChromeUIGpuCleanURL[] = "chrome://gpuclean/";
-inline constexpr char kChromeUIGpuCrashURL[] = "chrome://gpucrash/";
-inline constexpr char kChromeUIGpuHangURL[] = "chrome://gpuhang/";
-inline constexpr char kChromeUIHangURL[] = "chrome://hang/";
-inline constexpr char kChromeUIKillURL[] = "chrome://kill/";
-inline constexpr char kChromeUIMemoryExhaustURL[] = "chrome://memory-exhaust/";
+    "dao://delayeduithreadhang/";
+inline constexpr char kChromeUIDumpURL[] = "dao://crashdump/";
+inline constexpr char kChromeUIGpuCleanURL[] = "dao://gpuclean/";
+inline constexpr char kChromeUIGpuCrashURL[] = "dao://gpucrash/";
+inline constexpr char kChromeUIGpuHangURL[] = "dao://gpuhang/";
+inline constexpr char kChromeUIHangURL[] = "dao://hang/";
+inline constexpr char kChromeUIKillURL[] = "dao://kill/";
+inline constexpr char kChromeUIMemoryExhaustURL[] = "dao://memory-exhaust/";
 inline constexpr char kChromeUIMemoryPressureCriticalURL[] =
-    "chrome://memory-pressure-critical/";
+    "dao://memory-pressure-critical/";
 inline constexpr char kChromeUIMemoryPressureModerateURL[] =
-    "chrome://memory-pressure-moderate/";
+    "dao://memory-pressure-moderate/";
 inline constexpr char kChromeUINetworkErrorsListingURL[] =
-    "chrome://network-errors/";
-inline constexpr char kChromeUINetworkErrorURL[] = "chrome://network-error/";
+    "dao://network-errors/";
+inline constexpr char kChromeUINetworkErrorURL[] = "dao://network-error/";
 inline constexpr char kChromeUIProcessInternalsURL[] =
-    "chrome://process-internals/";
+    "dao://process-internals/";
 
 #if BUILDFLAG(IS_ANDROID)
-inline constexpr char kChromeUIGpuJavaCrashURL[] = "chrome://gpu-java-crash/";
+inline constexpr char kChromeUIGpuJavaCrashURL[] = "dao://gpu-java-crash/";
 #endif
-inline constexpr char kChromeUICrashRustURL[] = "chrome://crash/rust";
+inline constexpr char kChromeUICrashRustURL[] = "dao://crash/rust";
 #if BUILDFLAG(IS_WIN)
 inline constexpr char kChromeUIBrowserHeapCorruptionURL[] =
-    "chrome://inducebrowserheapcorruption/";
-inline constexpr char kChromeUICfgViolationCrashURL[] = "chrome://crash/cfg";
+    "dao://inducebrowserheapcorruption/";
+inline constexpr char kChromeUICfgViolationCrashURL[] = "dao://crash/cfg";
 inline constexpr char kChromeUIHeapCorruptionCrashURL[] =
-    "chrome://heapcorruptioncrash/";
+    "dao://heapcorruptioncrash/";
 #endif
 
 #if defined(ADDRESS_SANITIZER)
 inline constexpr char kChromeUICrashHeapOverflowURL[] =
-    "chrome://crash/heap-overflow";
+    "dao://crash/heap-overflow";
 inline constexpr char kChromeUICrashHeapUnderflowURL[] =
-    "chrome://crash/heap-underflow";
+    "dao://crash/heap-underflow";
 inline constexpr char kChromeUICrashUseAfterFreeURL[] =
-    "chrome://crash/use-after-free";
+    "dao://crash/use-after-free";
 #if BUILDFLAG(IS_WIN)
 inline constexpr char kChromeUICrashCorruptHeapBlockURL[] =
-    "chrome://crash/corrupt-heap-block";
+    "dao://crash/corrupt-heap-block";
 inline constexpr char kChromeUICrashCorruptHeapURL[] =
-    "chrome://crash/corrupt-heap";
+    "dao://crash/corrupt-heap";
 #endif  // BUILDFLAG(IS_WIN)
 inline constexpr char kChromeUICrashRustOverflowURL[] =
-    "chrome://crash/rust-overflow";
+    "dao://crash/rust-overflow";
 #endif  // ADDRESS_SANITIZER
 
 #if DCHECK_IS_ON()
-inline constexpr char kChromeUICrashDcheckURL[] = "chrome://crash/dcheck";
+inline constexpr char kChromeUICrashDcheckURL[] = "dao://crash/dcheck";
 #endif
 
 // Full about URLs (including schemes).
-inline constexpr char kChromeUIResourcesURL[] = "chrome://resources/";
-inline constexpr char kChromeUIShorthangURL[] = "chrome://shorthang/";
+inline constexpr char kChromeUIResourcesURL[] = "dao://resources/";
+inline constexpr char kChromeUIShorthangURL[] = "dao://shorthang/";
 
 // Returns whether the given url is either a debugging url handled in the
 // renderer process, such as one that crashes or hangs the renderer, or a
