diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 969c2af443..39a0fb2bc5 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -6495,6 +6495,8 @@ static_library("browser") {
       "//chrome/services/mac_notifications",
       "//chrome/services/mac_notifications/public/cpp",
       "//chrome/services/mac_notifications/public/mojom",
+      "//dao/browser/telemetry:dao_telemetry",
+      "//dao/browser/updater:dao_updater",
       "//components/crash/core/app",
       "//components/device_signals/core/browser/mac",
       "//components/enterprise/platform_auth:enterprise_platform_auth",
@@ -8307,7 +8309,7 @@ static_library("browser") {
 # all public deps of "browser" into a new target, and all circular dependencies
 # must include this target in a public_dep. This ensures stability and
 # correctness of the build-graph outside of //chrome/browser.
-static_library("browser_public_dependencies") {
+group("browser_public_dependencies") {
   public_deps = [
     # To avoid every target that is circularly dependent on
     # //chrome/browser:browser needing to include 2 separate targets, we include
@@ -8458,7 +8460,7 @@ static_library("browser_public_dependencies") {
 # //chrome/browser:browser should be listed here. This ensures that the build
 # graph is correct and that all headers are generated before any translation
 # units are compiled.
-static_library("browser_generated_files") {
+group("browser_generated_files") {
   public_deps = [
     ":buildflags",
     "//chrome/browser/media:media_engagement_preload_proto",
