diff --git a/chrome/browser/ui/webui/chrome_web_ui_configs.cc b/chrome/browser/ui/webui/chrome_web_ui_configs.cc
index acc4db4ef2..b224e231fb 100644
--- a/chrome/browser/ui/webui/chrome_web_ui_configs.cc
+++ b/chrome/browser/ui/webui/chrome_web_ui_configs.cc
@@ -5,6 +5,8 @@
 #include "chrome/browser/ui/webui/chrome_web_ui_configs.h"
 
 #include "build/build_config.h"
+#include "dao/browser/ui/webui/dao_agent_ui.h"
+#include "dao/browser/ui/webui/dao_sidebar_ui.h"
 #include "chrome/browser/optimization_guide/optimization_guide_internals_ui.h"
 #include "chrome/browser/ui/webui/about/about_ui.h"
 #include "chrome/browser/ui/webui/accessibility/accessibility_ui.h"
@@ -202,6 +204,11 @@ void RegisterChromeWebUIConfigs() {
   auto& map = content::WebUIConfigMap::GetInstance();
   map.AddWebUIConfig(std::make_unique<AccessibilityUIConfig>());
   map.AddWebUIConfig(std::make_unique<AutofillInternalsUIConfig>());
+
+  // Dao: Register WebUI pages
+  map.AddWebUIConfig(std::make_unique<dao::DaoAgentUIConfig>());
+  map.AddWebUIConfig(std::make_unique<dao::DaoSidebarUIConfig>());
+  map.AddWebUIConfig(std::make_unique<dao::DaoSkillsUIConfig>());
   map.AddWebUIConfig(std::make_unique<BluetoothInternalsUIConfig>());
   map.AddWebUIConfig(std::make_unique<BrowsingTopicsInternalsUIConfig>());
   map.AddWebUIConfig(std::make_unique<chromeos::DeviceLogUIConfig>());
