diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
index 2d39a00a30..2b5d7d45e1 100644
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -5,6 +5,9 @@
 #include "chrome/browser/ui/tab_helpers.h"
 
 #include <memory>
+#include "dao/browser/dao_auto_pip_visibility_helper.h"
+#include "dao/browser/dao_webstore_branding_tab_helper.h"
+#include "dao/browser/pip/dao_pip_interceptor.h"
 #include <utility>
 
 #include "base/command_line.h"
@@ -566,6 +569,9 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents) {
   vr::VrTabHelper::CreateForWebContents(web_contents);
   OneTimePermissionsTrackerHelper::CreateForWebContents(web_contents);
 
+  dao::DaoAutoPipVisibilityHelper::CreateForWebContents(web_contents);
+  dao::DaoPipInterceptor::CreateForWebContents(web_contents);
+  dao::DaoWebstoreBrandingTabHelper::CreateForWebContents(web_contents);
   // NO! Do not just add your tab helper here. This is a large alphabetized
   // block; please insert your tab helper above in alphabetical order.
 
