diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
index 6c3874b43f..bb7e63e93b 100644
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -6,6 +6,9 @@
 
 #include <cstdint>
 #include <memory>
+#include "dao/browser/dao_auto_pip_visibility_helper.h"
+#include "dao/browser/dao_webstore_branding_tab_helper.h"
+#include "dao/browser/pip/dao_pip_interceptor.h"
 #include <optional>
 #include <utility>
 
@@ -620,6 +623,9 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents) {
   vr::VrTabHelper::CreateForWebContents(web_contents);
   OneTimePermissionsTrackerHelper::CreateForWebContents(web_contents);
 
+  dao::DaoAutoPipVisibilityHelper::CreateForWebContents(web_contents);
+  dao::DaoPipInterceptor::CreateForWebContents(web_contents);
+  dao::DaoWebstoreBrandingTabHelper::CreateForWebContents(web_contents);
   // NO! Do not just add your tab helper here. This is a large alphabetized
   // block; please insert your tab helper above in alphabetical order.
 
