diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 43fd1559ee..e1cd4fa08a 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -585,6 +585,8 @@ source_set("extensions") {
     "//components/user_prefs",
     "//components/vector_icons",
     "//content/public/browser",
+    # Dao MV2 Support: bring in DaoMV2PrefDefaults for extension_management.cc.
+    "//dao/browser/extensions/legacy_mv2:dao_legacy_mv2",
     "//extensions/browser",
     "//extensions/browser:crx_installer",
     "//extensions/browser:cws_item_service_proto",
